The discovery of penicillin shortly before the Second World War revolutionized human medicine by saving the lives of millions of soldiers and civilians suffering under bacterial infections \cite{cdc_biggest_2019}. By introducing a treatment against bacterial infections, Penicillin also made major advances in surgery possible \cite{worldwar_resistance}. Since then antibiotics play a critical role in the health care system.\\
Because of the positive effects of penicillin, it quickly started to be used in excess and irresponsibly. This led to the emergence of penicillin resistance within 32 years after its discovery \cite{worldwar_resistance}.
Ever since then it has been necessary to repetitively introduce new antibiotics to the market, because bacteria sooner or later tend to evolve resistance to antibiotics in use. 

\section{Emergence of antibiotic resistance}
An emergence of antibiotic resistance is occurring worldwide \cite{ventola_antibiotic_2015}. An increase in antibiotic usage and a decrease of development in new antibiotics are the the main reasons for this emergence \cite{ventola_antibiotic_2015}. 
\begin{figure}[H]
	\includegraphics[width=1\textwidth]{consumption.png}
	\caption{A: Consumption of defined daily antibiotic doses per 1000 inhabitants per day over 15 years grouped by income.  B: Antibiotic consumption for selected countries associated with certain incomes. Especially India and other countries with a huge population with a lower income show a tremendous increase of antibiotic consumption \cite{klein_global_2018}. C: Decrease of FDA approved drugs between 1980 and 2014 \cite{ventola_antibiotic_2015}.}
	\label{figure:emergence}
\end{figure}
As shown in Figure \ref{figure:emergence} A and B, consumption mainly increased for countries where a huge population is associated with middle and lower income \cite{klein_global_2018}. In those two groups the increase between 2000 and 2015 was exponential. Countries where the average income is high show a rather stable consumption over the recorded years. However, the overall consumption is much higher compared to the other groups with middle and low income. This also suggests that there is an uneven access to antibiotics. \\
The number of antibiotics which were approved by the Food and Drug  Adminstration (FDA), on the other hand, decreased in the last four decades as shown in Figure \ref{figure:emergence} C. When between 1980 and 1984 19 antibiotics were approved by the FDA, only six antibiotics where approved between 2010 and 2014 \cite{ventola_antibiotic_2015}. The decrease in development in new antibiotics mainly has economical reasons. Recently approved antibiotics are usually held in reserve and only prescribed for infections that more established antibiotics cannot treat. This heavily limits the investment in return \cite{fair_antibiotics_2014}. Additionally, antibiotics are only used for a short period of time. Compared to drugs used for treating chronic diseases this makes antibiotics a lot less profitable  \cite{fair_antibiotics_2014}. The investment of return is further limited by the high costs of developing and approving a new antibiotic. Those costs are estimated as 0.5 billion US-Dollar \cite{costs}. \\
Antibiotics are used in two main areas. One being the health sector and one being the intensive animal farming. Unfortunately, in the health sector many antibiotics are prescribed unnecessarily. It was shown that 47 millions prescriptions of antibiotics in the USA were redundant \cite{noauthor_antibiotic_2019}. Most of them were prescribed for respiratory conditions most commonly caused by viruses \cite{noauthor_antibiotic_2019}. It is assumed that in the early 2000s 25-50\% of all antibiotic consumption took place in intense animal farming  \cite{palumbi_humans_2001}. Most of the antibiotics in intense animal farming are used for prophylaxis and for stimulating growth, but not for curing animals suffering infections. \\
The combination of increased usage and decreased development of antibiotics brought up resistance of all major bacterial pathogens. The most common resistant pathogens in Swiss hospitals are shown in Figure \ref{figure:pathogen_dvelopment}. The percentage of all resistant pathogens decreased or stayed the same for \textit{Staphylococci aurei}, \textit{Streptococci pneumoniae}, \textit{Enterococcus faecalis} and \textit{Enterococcus faecium}. This is ether because a vaccine was introduced (PCV7 vaccine against several \textit{Pneumococcal} strains) or screening of the pathogens was improved \cite{swiss_hospitals_pathogens}. Resistant \textit{Escherichia coli} (\textit{E. coli}) are becoming more common where mainly resistance against the antibiotics fluoroquinolones and extended-spectrum-\textbeta-lactams occurs. \\

\begin{figure}
	\includegraphics[width=0.7\textwidth]{pathogens_overview.png}
	\caption{Development of resistant pathogens in Swiss hospitals \cite{swiss_hospitals_pathogens}.}
	\label{figure:pathogen_dvelopment}
\end{figure}

\section{\textbeta-lactams and resistance against this class of antibiotics}
Different classes of antibiotics were developed during the 20th century. They target specific bacterial metabolic processes. The most common classes of antibiotics (black) and their targets (red) are shown in Figure \ref{figure:antibiotics}. \textbeta-lactams are by far the most commonly prescribed class of antibiotics \cite{blogger_guide_2019}.
\begin{figure}
	\includegraphics[width=0.68\textwidth]{antibiotics.jpg}
	\includegraphics[width=0.3\textwidth]{peptidoglycan.png}
	\caption{Left: Most popular classes of antibiotics and their targets in the bacterial cell \cite{wright_english:_2010}. Right structure of peptidocylcan \cite{illustrator_english:_nodate}}
	\label{figure:antibiotics}
\end{figure}

\subsection{\textbeta-lactams}
\subsubsection{Mode of action}
The \textbeta-lactam antibiotics inhibit the synthesis of the peptidoglycan layer of the bacterial cell wall \cite{noauthor_-lactam_2019}. A damaged cell wall leads to bursting of the cell caused by osmotic pressure. \\
The structure of peptidoglycan is shown in the right Figure \ref{figure:antibiotics}. It consists of polymers where N-acetylglucosamine (NAG) and N-acetylmuramic acid (NAM) alternates \cite{fisher_bacterial_2005}. A peptide with D-Alanyl-D-alanine as terminal residues is attached to NAMs \cite{fisher_bacterial_2005}. Those peptides are are cross-linked by DD-transpeptidases which gives the peptidoglycan its mesh-like structure \cite{fisher_bacterial_2005}. \textbeta-lactams are analogues of the amino acid D-Alanyl-D-alanine and bind to the active site of DD-transpeptidases and inhibit their activity irreversibly \cite{fisher_bacterial_2005}. 

\subsubsection{Cephalosporins}
The \textbeta-lactams contain a highly strained and reactive cyclic amide. Five groups of \textbeta-lactams exist: Penams, penems, carbapenems, monobactams and cefems \cite{beta-lactam_nodate}. 
Cephalosporins, belonging to cephems, are very important because they are very potent, well tolerated by patientsand widely used \cite{dancer_problem_2001}. Cephalosporins are divided into five generations.\\
\textbf{First-generation cephalosporins} are very active against gram-positive cocci but are not very potent against gram-negative bacteria \cite{fernandes_-lactams:_2013}. \textbf{The second-generation of cephalosporins} are all active against bacteria covered by the first-generation drugs, but have extended coverage against gram-negative bacteria \cite{fernandes_-lactams:_2013}. The enhanced activity against gram-negative bacteria is further visible in \textbf{third-generation cephalosporins}. A representative of this generation is ceftazidime \cite{klein_third-generation_1995}. The second- and third-generation of cephalosporins are also known as extended cephalosporins. Only two \textbeta-lactams are classified under the \textbf{fourth-generation of cephalosporins}, those being cefepim and cefpirome \cite{fernandes_-lactams:_2013}. The newest \textbf{fifth-generation of caphalosporins} were explicitly developed to target resistant strains \cite{fernandes_-lactams:_2013}.\\

\subsection{\textbeta-lactamases}
\label{section:resistance_mechanisms}
\textbeta-lactamases are enzymes which are able to hydrolyze \textbeta-lactams \cite{fisher_bacterial_2005}. There for they play an important role in the resistance mechanism of \textbeta-lactam resistant pathogens. 
Mainly plasmid-mediated \textbeta-lactamases are important for resistance because they can be transmitted via horizontal gene-transfer \cite{munita_mechanisms_2016}.  
The first plasmid-mediated \textbeta-lactamase in gram-negative bacteria was described in the early 1960s and called \textbeta-lactamase TEM-1  \cite{fernandes_-lactams:_2013}. TEM-1 was found worldwide only a few years later after its first isolation \cite{fernandes_-lactams:_2013}. TEM-1 caused resistance against \textbeta-lactams of this time.  This is why extended cephalosporins such as ceftazidime which couldn't be hydrolyzed by TEM-1 were developed \cite{fernandes_-lactams:_2013}. \\
Another known \textbeta-lactamase is called SHV-1, which was isolated for the first time in 1974 \cite{kuzin_structure_1999}. It is encoded chromosomally in the majority of isolates of \textit{K. pneumoniae}, but is also plasmid-mediated in \textit{E. coli} \cite{kuzin_structure_1999}. \\
Over time \textbeta-lactamses evolved which were able to hydrolyze extended-spectrum \textbeta-lactams and therefore called extended-sepctrum-\textbeta-lactamases (ESBL).  

\subsubsection{ESBL}
One ESBL is called CTX-M-1 \textbeta-lactamase, which was clinically isolated for the first time in Germany in 1986 \cite{bradford_extended-spectrum_2001}. Later on many variants of CTX-M-1 were isolated which were subdivided into five groups \cite{fernandes_-lactams:_2013}.
It is assumed that they evolved from the \textbeta-lactamase precursor AmpC from \textit{Klyzvera ascorbata}  \cite{bradford_extended-spectrum_2001}. Even though the first CTX-M-1 was isolated in Germany, it is mostly popular in eastern Europe, South America and Japan \cite{bradford_extended-spectrum_2001}. \\
Another ESBLs is called OXA.
The OXA family was originally created as a phenotypic rather than a genotypic group, based on a specific hydrolysis profile \cite{bradford_extended-spectrum_2001}. Its name comes from the ability to efficiently hydrolyze oxacillin \cite{bradford_extended-spectrum_2001}. \\

\subsection{General resistance mechanisms}
Generally there are three different strategies for a bacterial cell to reduce susceptibility against antibiotics. One strategy is to make it more difficult for the antibiotic to enter the cell which is effective because many antibiotics have target sites within the cell \cite{barreteau_cytoplasmic_2008}. The second strategy is to reduce the affinity or to protect the target site of the antibiotic. The last strategy is to  to reduce the amount of antibiotic molecules which are already within the cell. This is possible by degradation of the compound which we have seen with \textbeta-lactamases or by active transport out of the cell.\\
Some compounds such as \textbeta-lactams rely on channels in the membrane of the bacteria called porins in order to reach their target site within the cell. That is because e.g. \textbeta-lactams are hydrophilic and therefor can not just pass the lipophylic membrane by diffusion \cite{munita_mechanisms_2016}. Because antibiotics have very specific target sites another mechanism to gain resistance is to change the structure of the target \cite{munita_mechanisms_2016}. Some pathogens also protect their target sites by dislodging the compound by increasing the dissociation constant \cite{connell_ribosomal_2003}.
Active transport of antibiotics out of the cell is possible because bacterial cells are able to express substrate specific efflux pumps. Following this pathway is a widely spread mechanism against most classes of antibiotics \cite{munita_mechanisms_2016}.

\subsection{ESBL \textit{E. coli}}
\textit{E. coli} which have genes coding for ESBLs are called ESBL \textit{E. coli}. As seen in Figure \ref{figure:pathogen_dvelopment} infections with resistant ESBL \textit{E. coli} increased rapidly over the last 15 years. Not all ESBL \textit{E. coli} are resistant against extended-cephalosporins \cite{mcwilliams_incidence_2014}. Resistant ESBL \textit{E. coli} are susceptible against carbapenems \cite{fernandes_-lactams:_2013}. In order to limit further emergence of resistance, carbapenems should only be prescribed if the ESBL \textit{E. coli} strain is not susceptible to extended-cephalosporins . \\
The European Committee on Antimicrobial Susceptibility Testing (EUCAST) publishes breakpoints for most bacterial pathogen and antibiotic combinations. Those breakpoints advice clinicians after which minimal inhibitory concentration (MIC) of a certain antibiotic a pathogen is seen as resistant. They also publish guidelines which should help in choosing the appropriate antibiotic. 
In the past, when ESBL genes were detected by molecular screening in \textit{E. coli} the EUCAST recommended to prescribe last resort antibiotics no matter if the susceptibility of the ESBL \textit{E. coli} against extended-cephalosporins was below the breakpoint \cite{leclercq_eucast_2013}. The EUCAST changed their guidelines and only susceptibility testing determines which antibiotic is chosen for treatment \cite{leclercq_eucast_2013}. Unfortunately, the MIC determination takes about 48 hours. It would be of great benefit if it would be clear what determines if ESBL \textit{E. coli} are resistant to extended-cephalosporins on a genome level. Then genomic data could be used again for choosing the appropriate antibiotic speeding up the diagnostic process. \\ 

\section{Identifying mutations deciding over resistance or susceptibility of ESBL \textit{E. coli}}
ESBL genes alone do not determine, whether ESBL \textit{E. coli} are resistant to extended-cephalosporins or not. Instead it is possible that mutations, which have not been identified yet, play an important role in the resistance mechanism. Those mutations could be involved in other resistance mechanisms than \textbeta-lactam hydrolysis. \\
Such mutations can be identified by analyzing genomic data of extended-cephalosporin susceptible ESBL \textit{E. coli} which gained resistance by evolution. To get access to such genomic data, ESBL \textit{E. coli} can be sequenced which evolved resistance in a natural environment. It is also possible to analyze genomic data obtained from ESBL \textit{E. coli} which were forced to evolve resistance in vitro. Forcing evolution of resistance in vitro is possible with a device called morbidostat.

\subsection{Principles of the morbidostat} 
The morbidostat, originally invented by Toprak \textit{et al.}, is an automated culture device \cite{toprak_building_2013}. It allows to force bacteria to gain resistance by applying a constant high antibiotic pressure. With the morbidostat bacteria are grown in a fixed culture volume in vials. The growth in every vial is constantly monitored. Depending on how fast the bacteria grow an appropriate dose of antibiotics is injected into the vials. Culturing multiple days leads to evolution of resistance as phenotypes with increased resistance are constantly selected. \\
Several tasks are grouped in cycles and repetitively executed by the morbidostat. Over the defined cycle time the optical densities (ODs) are constantly measured which is represented with red dots in Figure \ref{figure:principle}. At the end of the cycle a fit is calculated approximating all the OD measurements from one cycle which is shown in in Figure \ref{figure:principle} as a black line going through the red dots. From that fit the growth rate is calculated. Also considering the growth of previous cycles, a feedback algorithm decides if and how much drug is injected in which vial. The calculated antibiotic concentration is injected with computer-controllable pumps. If liquids are injected, this also means that the cultures are diluted which is visible in the first OD measurement of the next cycle. As a last step of a cycle, volume which exceeds the culture volume is removed. As resistance emerges samples of the cultures grown in the morbidostat can be taken and sequenced.    
\begin{figure}
	%\includegraphics[width=0.5\textwidth]{vial_diagram.png}	
	\includegraphics[width=0.5\textwidth]{growth_diagram.png}
	\caption{OD measurements and their fit of three cycles of culturing with the morbidostat. }
	\label{figure:principle}
\end{figure} 

\subsection{Previous morbidostat experiments and expected outcome}
Dösselmann \textit{et al.} rebuilt the system in 2015  \cite{doselmann_rapid_2017} and used it for studying mutations associated with colistin resistance. With the morbidostat they were able to increase the MIC of colistin against \textit{Pseudomonas aeruginosa} 100-fold within 20 days \cite{doselmann_rapid_2017}. The mechanism of action for colistin is to displace cations from the phospate groups of membrane lipids. This leads to disruption of the outer cell membrane causing cell death \cite{noauthor_colistin:_nodate}. Therefore, it is no surprise that Dösselmann \textit{et al} could identify several mutations coding fro proteins involved in the lipopolysaccharide synthesis \cite{doselmann_rapid_2017}.

\subsection{Illumina and Nanopore sequencing}
Following evolution by studying genomic data relies on accurate sequencing data. Illumina sequencing is a method which produces short reads which are typically 150 base pairs (bp) long. The strong-suit of Illumina sequencing is the low error rate which was determined as 0.24 \% per base \cite{pfeiffer_systematic_2018}. A common used Illumina sequencing system is the MiSeq-System. Because the reads are only 150 bases long it is computationally not possible to assemble structurally correct whole-genomes. 
Oxford Nanopore Technologies produces very long reads which are up to several 100 kbp long. In contrast to Illumina sequencing the error rate per base is 13.6 \% \cite{noauthor_resolving_nodate} which is a lot higher. But because the reads are so long it is possible to assemble structurally correct whole-genomes. \\
By sequencing on both platforms it is possible to combine the benefits of the long reads of Nanopore with the low error rate of Illumina. This results in very accurate whole genome assemblies.

\section{Aim of this thesis}
The aim of this master thesis is to investigate the resistance evolution of ESBL \textit{E. coli} against the fourth-generation cephalosporin cefepime. The focus is on identifying mutations which accumulated in the genome of resistant ESBL \textit{E. coli} by evolutionary pathways. One part of achieving this aim relies on studying genomic data obtained from ESBL \textit{E. coli} samples, taken from patients, showing a change in their susceptibility against cefepime. The other part involves assembling a morbidostat and use it to force susceptible ESBL \textit{E. coli} to gain resistance. Similar to the first part, the aim is to follow the evolution of resistance by studying genomic data. The work involves
\begin{itemize}
	\item DNA sequencing with Oxford Nanopore Technologies
	\item De novo assembling of ESBL \textit{E. coli} combining Illumina and Nanopore Technology sequencing data
	\item Development of a bioinformatic analysis pipeline to identify mutations accumulated over time by evolutionary pathways
	\item Identification of mutations in resistant ESBL \textit{E. coli} patient isolates
	\item Assembling and troubleshooting of the morbidostat, use it to force ESBL \textit{E. coli} to evolve resistance
	\item Identification of mutations in resistant ESBL \textit{E. coli} where resistance evolution was forced with the morbidostat
	\item Providing annotation of the mutations
\end{itemize}  