The discovery of penicillin in 1928 by Alexander Fleming shortly before the Second World War revolutionized human medicine by saving the lives of thousands of soldiers and civilians suffering from bacterial infections \cite{cdc_biggest_2019}. Also by reducing the risk of infections, more complicated and advanced surgeries were possible \cite{worldwar_resistance}. 
For the discovery of penicillin Fleming received the novel price. In his award speech he warned people from irresponsible and excessive penicillin consumption which could lead in penicillin resistance:

 \say{The time may come when penicillin can be bought by anyone in the
shops. Then there is the danger that the ignorant man may easily underdose	himself and by exposing his microbes to non-lethal quantities of the drug
make them resistant \cite{fleming}}
 
 
Unfortunately Flemings prediction turned into reality and penicillin resistance emerged shortly after its discovery \cite{lobanovska_penicillins_2017}. Ever since then it has been necessary to repetitively introduce new antibiotics to the market, because bacteria sooner or later tend to evolve resistance to antibiotics in use. 

\section{Emergence of antibiotic resistance}
Antibiotic resistance is emerging worldwide \cite{ventola_antibiotic_2015}. An increase in antibiotic usage and a decrease of development in new antibiotics are the the main reasons for this emergence \cite{ventola_antibiotic_2015}. 
\begin{figure}
	\includegraphics[width=1\textwidth]{consumption.png}
	\caption{A: Consumption of defined daily antibiotic doses per 1000 inhabitants per day over 15 years grouped by income.  B: Antibiotic consumption for selected countries associated with certain incomes. Especially India and other countries with a huge population with a lower income show a tremendous increase of antibiotic consumption \cite{klein_global_2018}. C: Decrease of FDA approved drugs between 1980 and 2014 \cite{ventola_antibiotic_2015}.}
	\label{figure:emergence}
\end{figure}
As shown in Figure \ref{figure:emergence} A and B, consumption mainly increased for countries where a huge population is associated with middle and lower income \cite{klein_global_2018}. Countries where the average income is high show a rather stable consumption over the recorded years. However, the overall consumption is much higher compared to the other groups with middle and low income. This also suggests that there is an uneven access to antibiotics. \\
The number of antibiotics which were approved by the Food and Drug  Adminstration (FDA), on the other hand, decreased in the last four decades as shown in Figure \ref{figure:emergence} C. When between 1980 and 1984 19 antibiotics were approved by the FDA, only six antibiotics where approved between 2010 and 2014 \cite{ventola_antibiotic_2015}. The decrease in development in new antibiotics mainly has economical reasons. Recently approved antibiotics are usually held in reserve and only prescribed for infections that more established antibiotics cannot treat. This heavily limits the investment of return \cite{fair_antibiotics_2014}. Additionally, antibiotics are only used for a short period of time. Compared to drugs used for treating chronic diseases this makes antibiotics a lot less profitable  \cite{fair_antibiotics_2014}. The investment of return is further limited by the high costs of developing and approving a new antibiotic. Those costs are estimated as 0.5 billion US-Dollar \cite{costs}. \\
Antibiotics are used in two main areas. One being the health sector and one being the intensive animal farming. Unfortunately, in the health sector many antibiotics are prescribed unnecessarily. It was shown that 47 millions prescriptions of antibiotics in the USA were redundant \cite{noauthor_antibiotic_2019}. Most of them were prescribed for respiratory conditions most commonly caused by viruses \cite{noauthor_antibiotic_2019}. It is assumed that in the early 2000s 25-50\% of all antibiotic consumption took place in intense animal farming  \cite{palumbi_humans_2001}. Most of the antibiotics in intense animal farming are used for prophylaxis and for stimulating growth, but not for curing animals suffering infections. \\
The combination of increased usage and decreased development of antibiotics brought up resistance of all major bacterial pathogens. The most common resistant pathogens in Swiss hospitals are shown in Figure \ref{figure:pathogen_dvelopment}. The percentage of all resistant pathogens decreased or stayed the same for \textit{Staphylococci aurei}, \textit{Streptococci pneumoniae}, \textit{Enterococcus faecalis} and \textit{Enterococcus faecium}. This is either because a vaccine was introduced (PCV7 vaccine against several \textit{Pneumococcal} strains) or screening of the pathogens was improved \cite{swiss_hospitals_pathogens}. Resistant \textit{Escherichia coli} (\textit{E. coli}) are becoming more common where mainly resistance against the antibiotics fluoroquinolones and extended-spectrum-\textbeta-lactams occurs. 

Different classes of antibiotics were developed during the 20th century. They target specific bacterial metabolic processes. The most common classes of antibiotics (black) and their targets (red) are shown in Figure \ref{figure:antibiotics}. \textbeta-lactams are the most commonly prescribed class of antibiotics \cite{blogger_guide_2019}.

\section{\textbeta-lactams}
\begin{figure}
	\includegraphics[width=0.68\textwidth]{antibiotics.jpg}
	\includegraphics[width=0.3\textwidth]{peptidoglycan.png}
	\caption{Left: Most popular classes of antibiotics and their targets in the bacterial cell \cite{wright_english:_2010}. Right structure of peptidocylcan \cite{illustrator_english:_nodate}}
	\label{figure:antibiotics}
\end{figure}

\subsection{Mode of action}
The \textbeta-lactam antibiotics inhibit the synthesis of the peptidoglycan layer of the bacterial cell wall \cite{noauthor_-lactam_2019}. A damaged cell wall leads to bursting of the cell caused by osmotic pressure. \\
The structure of peptidoglycan is shown in the right Figure \ref{figure:antibiotics}. 
The monosaccharides  N-acetylglucosamine (NAG) and N-acetylmuramic acid (NAM) are alternately linked forming polysaccharides which are the backbone of the peptidoglycan \cite{fisher_bacterial_2005}.
A peptide with D-Alanyl-D-alanine as terminal residues is attached to NAMs \cite{fisher_bacterial_2005}. Those peptides are cross-linked by DD-transpeptidases which gives the peptidoglycan its mesh-like structure \cite{fisher_bacterial_2005}. \textbeta-lactams are analogues of the amino acid D-Alanyl-D-alanine and bind to the active site of DD-transpeptidases which causes an irreversible inhibition of their activity \cite{fisher_bacterial_2005}. 

\subsection{Cephalosporins}
\begin{figure}
	\includegraphics[width=0.7\textwidth]{pathogens_overview.png}
	\includegraphics[width=0.25\textwidth]{cefepim.png}
	\caption{Left: Development of resistant pathogens in Swiss hospitals \cite{swiss_hospitals_pathogens}. Right: Structure of cefepime, a fourth-generation cephalosporin \cite{noauthor_datei:cefepim_nodate}.}
	\label{figure:pathogen_dvelopment}
	\label{figure:cefepim}
\end{figure}
The \textbeta-lactams contain a highly strained and reactive cyclic amide. Five groups of \textbeta-lactams exist: Penams, penems, carbapenems, monobactams and cefems \cite{beta-lactam_nodate}. 
Cephalosporins, belonging to cephems, are very important because they are very potent, well tolerated by patients and thus widely used \cite{dancer_problem_2001}. 
All cephalosporins share an identical core structure but have two varying residues connected \cite{noauthor_cephalosporine_2019}. The core structure is shown in the right Figure \ref{figure:cefepim} in black. The residues shown in blue and green are the residues of the cephalosporin cefepime. Five generations of cephalosporins exist.
\textbf{First-generation cephalosporins} are very active against gram-positive cocci but are not very potent against gram-negative bacteria \cite{fernandes_-lactams:_2013}. \textbf{The second-generation of cephalosporins} are all active against bacteria covered by the first-generation drugs, but have extended coverage against gram-negative bacteria \cite{fernandes_-lactams:_2013}. The enhanced activity against gram-negative bacteria is further visible in \textbf{third-generation cephalosporins}. A representative of this generation is ceftazidime \cite{klein_third-generation_1995}. The second- and third-generation of cephalosporins are also known as extended cephalosporins. Only two \textbeta-lactams are classified under the \textbf{fourth-generation of cephalosporins}, those being cefepim and cefpirome \cite{fernandes_-lactams:_2013}. The newest \textbf{fifth-generation of caphalosporins} ceftobiprole and ceftaroline  were explicitly developed to target multi resistant \textit{Staphylococcus aureus} \cite{fernandes_-lactams:_2013}.

\section{Resistance mechanisms}
Generally there are three different strategies for a bacterial cell to reduce susceptibility against antibiotics which have targets within the cell. One strategy is to reduce the uptake of antibiotics \cite{barreteau_cytoplasmic_2008}. The second strategy is to reduce the affinity or to protect the target site of the antibiotic \cite{munita_mechanisms_2016}. The last strategy is to  to reduce the antibiotic concentration in the cell \cite{munita_mechanisms_2016}.\\
Uptake of antibiotics occurs commonly through water-filled membrane proteins called porins \cite{fernandez_adaptive_2012}. Bacterial cells can limit the uptake of antibiotics through porins by reducing the expression level, or changing their size \cite{fernandez_adaptive_2012}.
Reducing the affinity of the target sites is achieved by point mutations in the gene coding for the target, or by enzymatic alteration of the target site (e.g addition of methyl groups) \cite{munita_mechanisms_2016}. The affinity can also be reduced by disloding the antibiotic which increases the dissociation constant \cite{connell_ribosomal_2003}.
Reduction of the antibiotic concentration is achieved by active transport or by degradation. Active transport involves efflux pumps. Those pumps transport the compound through energy-dependent membrane proteins out of the cell \cite{fernandez_adaptive_2012}. Degradation of antibiotics, usually by hydrolysis, is an enzymatic process and plays an important role in resistance against \textbeta-lactams \cite{poole_resistance_2004}.  
\subsection{Resistance against \textbeta-lactams}
Resistance against \textbeta-lactams is mostly achieved by expressing enzymes called \textbeta-lactamases which are able to hydrolyze \textbeta-lactam \cite{poole_resistance_2004}. For some bacterial strains target muations in the DD-transpeptidases have been reported which were also associated with \textbeta-lactam resistance \cite{poole_resistance_2004}. Since \textbeta-lactams are hydrophilic, they pass the outer membrane through porins \cite{munita_mechanisms_2016}. Therefore, porins have been reported to play an important role in the resistance mechanism against \textbeta-lactams as well \cite{poole_resistance_2004}.
\subsubsection{\textbeta-lactamases}
\label{section:resistance_mechanisms}
Mainly plasmid-mediated \textbeta-lactamases are important for resistance because they can be transmitted via horizontal gene-transfer \cite{munita_mechanisms_2016}.  
The first plasmid-mediated \textbeta-lactamase in gram-negative bacteria was described in the early 1960s and called \textbeta-lactamase TEM-1  \cite{fernandes_-lactams:_2013}. TEM-1 was found worldwide only a few years later after its first isolation \cite{fernandes_-lactams:_2013}. TEM-1 caused resistance against \textbeta-lactams of this time.  This is why extended cephalosporins such as ceftazidime which couldn't be hydrolyzed by TEM-1 were developed \cite{fernandes_-lactams:_2013}. \\
Another known \textbeta-lactamase is called SHV-1, which was isolated for the first time in 1974 \cite{kuzin_structure_1999}. It is encoded chromosomally in the majority of isolates of \textit{K. pneumoniae}, but is also plasmid-mediated in \textit{E. coli} \cite{kuzin_structure_1999}. \\
Over time \textbeta-lactamses evolved which were able to hydrolyze extended-spectrum \textbeta-lactams and therefore called extended-spectrum-\textbeta-lactamases (ESBL).  

\subsubsection{ESBL}
One ESBL is called CTX-M-1 \textbeta-lactamase, which was clinically isolated for the first time in Germany in 1986 \cite{bradford_extended-spectrum_2001}. Later on many variants of CTX-M-1 were isolated which were subdivided into five groups \cite{fernandes_-lactams:_2013}.
It is assumed that they evolved from the \textbeta-lactamase precursor AmpC from \textit{Klyzvera ascorbata}  \cite{bradford_extended-spectrum_2001}. Even though the first CTX-M-1 was isolated in Germany, it is mainly present in South America and Japan \cite{bradford_extended-spectrum_2001}. \\
Another ESBLs is called OXA.
The OXA family was originally created as a phenotypic rather than a genotypic group, based on a specific hydrolysis profile \cite{bradford_extended-spectrum_2001}. Its name comes from the ability to efficiently hydrolyze oxacillin \cite{bradford_extended-spectrum_2001}. \\

\section{ESBL \textit{E. coli}}
\textit{E. coli} belonging to the gram-negative bacteria are commonly present in the human intestine \cite{swiss_hospitals_pathogens}. If present in other parts of the body \textit{E. coli} can cause infections \cite{swiss_hospitals_pathogens}.
\textit{E. coli} strains expressing ESBLs are called ESBL \textit{E. coli}. As seen in the left Figure \ref{figure:pathogen_dvelopment} infections with resistant ESBL \textit{E. coli} increased rapidly over the last 15 years. 

\subsection{Identification ESBL \textit{E. coli} and treatment}
The European Committee on Antimicrobial Susceptibility Testing (EUCAST) is an organization advising clinicians which actions should be taken in response to results from  antimicrobial susceptibility testing (AST) \cite{leclercq_eucast_2013}. For \textit{E. coli} where clinicians suspect the presence of ESBLs, the EUCAST recommends to confirm ESBLs by a standardized ESBL disc test assay \cite{hombach_consequences_2013}. For this assay, \textit{E. coli} are grown with a fixed density on a plate \cite{disc_test}. On this plate discs with certain cefotaxime, ceftazidime and cefepime alone, but also in combination with the ESBL inhibitor clavulanic acid, are placed \cite{disc_test}. After an incubation time of 18-24 hours the diameter of the inhibition zone on the plates is measured \cite{disc_test}. If the diameter of the inhibition zones, caused by the antibiotic and clavulanic acid combination, is bigger than 5 mm the tested \textit{E. coli} strain is seen as ESBL producing \cite{disc_test}. In those cases clinicians are recommended to use carbapenems for treatment \cite{mcwilliams_incidence_2014}. This procedure where ESBL detection is only based on AST was introduced in 2013. Before, the EUCAST advised to detect ESBLs by gene screening with molecular techniques (e.g PCR, microarrays) \cite{screening}. If genes coding for ESBLs were found carbapenems were prescribed. Interestingly many \textit{E. coli} strains screened positive for ESBL genes were susceptible against common cephalosporins \cite{hombach_consequences_2013}.
Hombach \textit{et al.} showed that 32.5\% of all \textit{E. coli} isolates that he studied, screened positive for the gene coding for CTX-M, were susceptible to cefepime \cite{hombach_consequences_2013}. This has led to many cases where \textit{E. coli} strains with ESBL genes were treated unnecessary with carbapenems. This is why the EUCAST changed the guidelines in 2013 and now only AST is considered. In principal it would be helpful for clinicians to include genomic data again, because AST takes very long (about 48 hours) and is not very accurate \cite{disc_test}\cite{hombach_consequences_2013}.

\section{Identifying mutations predicting resistance or susceptibility of ESBL \textit{E. coli}}
ESBL genes alone do not determine, whether ESBL \textit{E. coli} are resistant to cephalosporins or not. Instead it is possible that mutations, which have not been identified yet, play an important role in the resistance mechanism. Those mutations could affect the expression level of ESBLs or increase the membrane impermeability by changing the structure of the porins. Furthermore, efflux pumps could be affected by mutations causing an active transport of \textbeta-lactams out of the cell or the affinity of the DD-transpeptidases could be decreased.
Such mutations can be identified by analyzing genomic data of extended-cephalosporin susceptible ESBL \textit{E. coli} which gained resistance by evolution. To get access to such genomic data, the same strain of ESBL \textit{E. coli} can be sequenced before and after resistance emerged. This requires access to a clinical sample collection where susceptibility changed over time. It is also possible to analyze genomic data obtained from ESBL \textit{E. coli} which were forced to evolve resistance in vitro. Studying resistance with experimental evolution is possible with a device called morbidostat.

\subsection{Principles of the morbidostat} 
The morbidostat, originally invented by Toprak \textit{et al.}, is an automated culture device \cite{toprak_building_2013}. It allows to force bacteria to gain resistance by applying a constant high antibiotic pressure. With the morbidostat bacteria are grown in a fixed culture volume in vials. The growth in every vial is constantly monitored. Depending on how fast the bacteria grow an appropriate dose of antibiotics is injected into the vials. Culturing multiple days leads to evolution of resistance as phenotypes with increased resistance are constantly selected. \\
Several tasks are grouped in cycles and repetitively executed by the morbidostat. Over the defined cycle time the optical densities (ODs) are constantly measured which is represented with red dots in Figure \ref{figure:principle}. At the end of the cycle a fit is calculated approximating all the OD measurements from one cycle which is shown in in Figure \ref{figure:principle} as a black line going through the red dots. From that fit the growth rate is calculated. Also considering the growth of previous cycles, a feedback algorithm decides if and how much drug is injected in which vial. The calculated antibiotic concentration is injected with computer-controllable pumps. If liquids are injected, this also means that the cultures are diluted which is visible in the first OD measurement of the next cycle. As a last step of a cycle, volume which exceeds the culture volume is removed. As resistance emerges samples of the cultures grown in the morbidostat can be taken and sequenced.    
\begin{figure}
	%\includegraphics[width=0.5\textwidth]{vial_diagram.png}	
	\includegraphics[width=0.5\textwidth]{growth_diagram.png}
	\caption{OD measurements and their fit of three cycles of culturing with the morbidostat. }
	\label{figure:principle}
\end{figure} 

\subsection{Previous morbidostat experiments and expected outcome}
Dösselmann \textit{et al.} rebuilt the system in 2015  \cite{doselmann_rapid_2017} and used it for studying mutations associated with colistin resistance. With the morbidostat they were able to increase the MIC of colistin against \textit{Pseudomonas aeruginosa} 100-fold within 20 days \cite{doselmann_rapid_2017}. The mechanism of action for colistin is to displace cations from the phospate groups of membrane lipids. This leads to disruption of the outer cell membrane causing cell death \cite{noauthor_colistin:_nodate}. Therefore, it is no surprise that Dösselmann \textit{et al} could identify several mutations coding for proteins involved in the lipopolysaccharide synthesis \cite{doselmann_rapid_2017}.

\subsection{Illumina and Nanopore sequencing}
Following evolution by studying changes in the genome relies on accurate sequencing data. Illumina sequencing is a method which produces short reads which are typically 150 base pairs (bp) long. The strong-suit of Illumina sequencing is the low error rate which was determined as 0.24 \% per base \cite{pfeiffer_systematic_2018}. A commonly used Illumina sequencing system is the MiSeq-System. Because the reads are only 150 bases long it is computationally difficult to assemble whole-genomes because overlapping regions are small.  
Oxford Nanopore Technologies (ONT) produces very long reads which are up to several 100 kbp long. In contrast to Illumina sequencing the error rate per base is 13.6 \% which is a lot higher \cite{noauthor_resolving_nodate}. Most of those errors are caused by repetitive regions in the genome. ONT has issues in detecting those regions correctly \cite{noauthor_resolving_nodate}. False nucleotide substitutions are only found at a rate of 3\%. The long reads allow to assemble structurally correct whole-genomes. \\
By sequencing on both platforms it is possible to combine the benefits of the long reads of Nanopore with the low error rate of Illumina. This results in very accurate whole genome assemblies.

\section{Aim of this thesis}
The aim of this master thesis is to investigate the resistance evolution of ESBL \textit{E. coli} against the fourth-generation cephalosporin cefepime. The focus is on identifying mutations which accumulated in the genome of resistant ESBL \textit{E. coli} by evolutionary pathways. One part of achieving this aim relies on studying genomic data obtained from ESBL \textit{E. coli} samples, taken from patients, showing a change in their susceptibility against cefepime. The other part involves assembling a morbidostat and use it to force susceptible ESBL \textit{E. coli} to gain resistance. Similar to the first part, the aim is to follow the evolution of resistance by studying genomic data. The work involves
\begin{itemize}
	\item DNA sequencing with Oxford Nanopore Technologies
	\item De novo assembling of ESBL \textit{E. coli} combining Illumina and Nanopore Technology sequencing data
	\item Assembling and troubleshooting of the morbidostat, use it to experimentally evolve resistance to cefepime with ESBL \textit{E. coli} and cefepime
	\item Development of a bioinformatic analysis pipeline to identify mutations accumulated over time by evolutionary pathways
	\item Identification of mutations in resistant ESBL \textit{E. coli} patient isolates
	\item Identification of mutations in resistant ESBL \textit{E. coli} where resistance evolved experimentally with the morbidostat
\end{itemize}  