The discovery of penicillin in 1928 by Alexander Fleming shortly before the Second World War revolutionized human medicine by saving the lives of thousands of soldiers and civilians suffering from bacterial infections \cite{cdc_biggest_2019}. Fleming received the Nobel Prize for the discovery of penicillin \cite{fleming}. In his award speech he warned people from irresponsible and excessive penicillin consumption which could lead to penicillin resistance:

 \say{The time may come when penicillin can be bought by anyone in the
shops. Then there is the danger that the ignorant man may easily underdose	himself and by exposing his microbes to non-lethal quantities of the drug
make them resistant \cite{fleming}}
 
 
Unfortunately, Flemings prediction came true and penicillin resistance emerged shortly after its discovery \cite{lobanovska_penicillins_2017}. Ever since then it has been necessary to repetitively introduce new antibiotics to the market, because bacteria sooner or later tend to evolve resistance to antibiotics in use. 

\section{Emergence of antibiotic resistance}
Antibiotic resistance is emerging worldwide \cite{ventola_antibiotic_2015}. An increase in antibiotic consumption and a decrease in the development of new antibiotics are the main reasons for the emergence of antibiotic resistance \cite{ventola_antibiotic_2015}. 
\begin{figure}
	\includegraphics[width=1\textwidth]{consumption.png}
	\caption{A: Worldwide consumption of defined daily antibiotic doses per 1000 inhabitants per day, grouped by income \cite{klein_global_2018}.  B: Billions of defined daily doses for selected countries representing different income groups \cite{klein_global_2018}. C: Decrease of FDA approved drugs between 1980 and 2014 \cite{ventola_antibiotic_2015}.}
	\label{figure:emergence}
\end{figure}
As shown in Figure \ref{figure:emergence} A and B, consumption mainly increased for countries where a huge population is associated with middle and lower income. Countries with high average income show a rather stable consumption over the recorded years. However, the overall consumption is much higher compared to the countries with middle and low income \cite{klein_global_2018}. This also suggests that the access to antibiotics is uneven.\\
The number of antibiotics which were approved by the Food and Drug  Adminstration (FDA) decreased in the last four decades as shown in Figure \ref{figure:emergence} C \cite{ventola_antibiotic_2015}. Whereas between 1980 and 1984 19 antibiotics were approved by the FDA, only six antibiotics were approved between 2010 and 2014 \cite{ventola_antibiotic_2015}. The decrease in development in new antibiotics mainly has economical reasons. Recently approved antibiotics are usually held in reserve and only prescribed for infections that already established antibiotics cannot treat. This limits the investment of return \cite{fair_antibiotics_2014}. Additionally, antibiotics are only used for a short period of time. This makes antibiotics a lot less profitable compared to drugs used for treating chronic diseases or cancer \cite{fair_antibiotics_2014}. The investment of return is further limited by the high costs of developing and approving a new antibiotic which is estimated to be 0.5 billion US-Dollar \cite{costs}. \\
Antibiotics are used in two main areas: The health sector and intensive animal farming. Unfortunately, in the health sector many antibiotics are prescribed unnecessarily. For instance, it was shown that in 2010 47 millions prescriptions of antibiotics in the USA were redundant \cite{noauthor_antibiotic_2019}. Most of them were prescribed for respiratory conditions most commonly caused by viruses \cite{noauthor_antibiotic_2019}. It is assumed that in the early 2000s 25-50\% of all antibiotic consumption took place in intensive animal farming  \cite{palumbi_humans_2001}. Most of the antibiotics in intensive animal farming are used as prophylaxis and for stimulating growth, not for curing animals suffering from infections \cite{palumbi_humans_2001}\cite{zhu_diverse_2013}. \\
The combination of increased usage and decreased development of antibiotics has caused resistance in all major bacterial pathogens. The most common resistant pathogens in Swiss hospitals are shown in Figure \ref{figure:pathogen_dvelopment}. The percentage of all resistant pathogens decreased or remained consistent for \textit{Staphylococcus aureus}, \textit{Streptococcus pneumoniae}, \textit{Enterococcus faecalis} and \textit{Enterococcus faecium}. \textit{Escherichia coli} (\textit{E. coli}) are becoming more common where mainly resistance against the antibiotics fluoroquinolones and extended-spectrum cephalosporins occurs. 

Different classes of antibiotics were developed during the 20th century. They target specific bacterial metabolic processes. The most common classes of antibiotics (black) and their targets (red) are shown in Figure \ref{figure:antibiotics}.
\begin{figure}
	\includegraphics[width=0.5\textwidth]{pathogens_overview.png}
	\includegraphics[width=0.5\textwidth]{antibiotics.jpg}
	\caption{Left: Development of antibiotic resistance among pathogens in Swiss hospitals \cite{swiss_hospitals_pathogens}. Right: Most popular classes of antibiotics and their targets in the bacterial cell \cite{wright_english:_2010}.}
	\label{figure:pathogen_dvelopment}
\end{figure}
\newpage
\section{\textbeta-lactams}
\begin{figure}[b]
	\includegraphics[width=0.5\textwidth]{peptidoglycan.png}
	\includegraphics[width=0.4\textwidth]{cefepim.png}
	\caption{Left: Structure of peptidocylcan \cite{noauthor_bacteria:_nodate}. Cross-linked peptides attached to the polysaccharides cause a mesh-like structure of the peptidoglycan. Right: Structure of cefepime, a fourth-generation cephalosporin \cite{noauthor_datei:cefepim_nodate}. The structure in black stands for the core structure shared by every cephalosporin. Structures colored in green and blue are the residues of cefepime.}
	\label{figure:antibiotics}
	\label{figure:cefepim}
\end{figure}
\subsection{Mode of action}
The \textbeta-lactam antibiotics inhibit the synthesis of the peptidoglycan layer of the bacterial cell wall \cite{fisher_bacterial_2005}. A damaged cell wall leads to bursting of the cell caused by osmotic pressure. \\
The structure of peptidoglycan is shown in Figure \ref{figure:antibiotics}. 
The monosaccharides  N-acetylglucosamine (NAG) and N-acetylmuramic acid (NAM) are alternately linked forming polysaccharides which are the backbone of the peptidoglycan \cite{fisher_bacterial_2005}.
A peptide with D-alanyl-D-alanine as terminal residues is attached to NAMs \cite{fisher_bacterial_2005}. Those peptides are cross-linked by DD-transpeptidases which gives the peptidoglycan its mesh-like structure \cite{fisher_bacterial_2005}. \textbeta-lactams are analogues of the amino acid D-alanyl-D-alanine and bind to the active site of DD-transpeptidases which causes an irreversible inhibition of their activity \cite{fisher_bacterial_2005}. 

\subsection{Cephalosporins}
The \textbeta-lactams contain a highly strained and reactive cyclic amide. Five groups of \textbeta-lactams exist: Penams, penems, carbapenems, monobactams and cephems \cite{beta-lactam_nodate}. 
Cephalosporins, belonging to cephems, are very important because they are very potent, well tolerated by patients and thus widely used \cite{dancer_problem_2001}. 
All cephalosporins share an identical core structure which is shown in black in Figure \ref{figure:cefepim}. Different residues attached to the core structure are determining the specific cephalosporin. Five generations of cephalosporins exist.
\textbf{First-generation cephalosporins} are very active against gram-positive cocci but are not very potent against gram-negative bacteria \cite{fernandes_-lactams:_2013}. The \textbf{second-generation cephalosporins} are all active against bacteria covered by the first-generation drugs, but have extended activity against gram-negative bacteria \cite{fernandes_-lactams:_2013}. The enhanced activity against gram-negative bacteria is further visible in \textbf{third-generation cephalosporins}. A representative of this generation is ceftazidime \cite{klein_third-generation_1995}. Only two \textbeta-lactams are classified as \textbf{fourth-generation cephalosporins}: Cefepime and cefpirome \cite{fernandes_-lactams:_2013}. The residues of cefepime are shown in green and blue in Figure \ref{figure:cefepim}. The newest \textbf{fifth-generation of caphalosporins} ceftobiprole and ceftaroline  were explicitly developed to target multi resistant \textit{Staphylococcus aureus} \cite{fernandes_-lactams:_2013}. Cephalosporins belonging to second or later generations are also referred as extended-spectrum cephalosporins.

\section{Resistance mechanisms}
Generally, there are three different strategies for bacteria to reduce susceptibility against antibiotics which have targets within the cell such as cephalosporins. One strategy is to reduce the uptake of antibiotics \cite{barreteau_cytoplasmic_2008}. Uptake of antibiotics occurs commonly through water-filled membrane proteins called porins \cite{fernandez_adaptive_2012}. Bacterial cells can limit the uptake of antibiotics by reducing the expression level of porins, or by changing their size \cite{fernandez_adaptive_2012}. The second strategy is to reduce the affinity or to protect the target site of the antibiotic \cite{munita_mechanisms_2016}. Reducing the affinity of the target sites is achieved by point mutations in the gene coding for the target, or by enzymatic alteration of the target site (\textit{e.g} addition of methyl groups) \cite{munita_mechanisms_2016}. Affinity can be reduced because other ligands bind to the target and are dislodging the antibiotic from the target which increases the dissociation constant of the antibiotic \cite{connell_ribosomal_2003}. The last strategy is to reduce the antibiotic concentration in the cell \cite{munita_mechanisms_2016}. Reduction of the antibiotic concentration is achieved by active transport or by degradation. Active transport involves efflux pumps. Those pumps transport the compound through energy-dependent membrane proteins out of the cell \cite{fernandez_adaptive_2012}. Degradation of antibiotics, usually by hydrolysis, is an enzymatic process and plays an important role in resistance against cephalosporins \cite{poole_resistance_2004}.  
\subsection{Resistance against \textbeta-lactams}
Resistance to \textbeta-lactams is mainly associated with decreased antibiotic concentration in the cell caused by hydrolysis of \textbeta-lactams. Hydrolysis of \textbeta-lactams is mediated by enzymes called \textbeta-lactamases \cite{poole_resistance_2004}. For some bacterial strains target mutations in the DD-transpeptidases have been reported which were also associated with \textbeta-lactam resistance \cite{poole_resistance_2004}. Becuase of the hydrophilic property \textbeta-lactams they can not pass the lipid dominated outer membrane by diffusion \cite{poole_resistance_2004}. In order to pass the outer membrane \textbeta-lactams depend on water filled membrane proteins called porins \cite{fernandez_adaptive_2012}. The dependency on porins in order to reach the target site in the cell gives the bacterial the advantage to decrease the permeability of the outer membrane by reducing the expression of porins, or by changing their structure leading to a reduced \textbeta-lactam uptake \cite{poole_resistance_2004}. Therefore, porins have been reported to play an important role in the resistance mechanism against \textbeta-lactams, because they effectively can reduce the uptake of \textbeta-lactams\cite{poole_resistance_2004}.
\subsubsection{\textbeta-lactamases}
\label{section:resistance_mechanisms}
Plasmid-mediated \textbeta-lactamases are important for resistance because they can be transmitted via horizontal gene-transfer \cite{munita_mechanisms_2016}.  
The first plasmid-mediated \textbeta-lactamase in gram-negative bacteria was described in the early 1960s and called \textbeta-lactamase TEM-1  \cite{fernandes_-lactams:_2013}. TEM-1 was found worldwide only a few years later after its first isolation \cite{fernandes_-lactams:_2013}. TEM-1 caused resistance against \textbeta-lactams of this time.  This is why extended-spectrum cephalosporins such as ceftazidime which could not be hydrolyzed by TEM-1 were developed \cite{fernandes_-lactams:_2013}. Another known \textbeta-lactamase is called SHV-1, which was isolated for the first time in 1974 \cite{kuzin_structure_1999}. It is encoded chromosomally in the majority of isolates of \textit{K. pneumoniae}, but is also transmitted via plasmids in \textit{E. coli} \cite{kuzin_structure_1999}. \\

\subsubsection{Extended-spectrum \textbeta-lactamase (ESBL)}
Over time \textbeta-lactamses evolved and were able to hydrolyze extended-spectrum cephalosporins. Those novel \textbeta-lactamases were called extended-spectrum-\textbeta-lactamases (ESBL).  
CTX-M-1 is an ESBL, which was clinically isolated for the first time in Germany in 1986 \cite{bradford_extended-spectrum_2001}. Many variants of CTX-M-1 were isolated and they were divided in five groups \cite{fernandes_-lactams:_2013}.
It is assumed that they evolved from the \textbeta-lactamase precursor AmpC from \textit{Klyzvera ascorbata}  \cite{bradford_extended-spectrum_2001}. 
OXA is another common ESBL family. Originally the OXA family was created as a phenotypic rather than a genotypic group, based on a specific hydrolysis profile \cite{bradford_extended-spectrum_2001}. Its name comes from the ability to efficiently hydrolyze oxacillin \cite{bradford_extended-spectrum_2001}. \\

\section{ESBL \textit{E. coli}}
The most common ESBL-producing bacteria are \textit{E. coli} and \textit{Klebsiella pneumoniae}. \textit{E. coli} is a gram-negative bacteria and is commonly present in the human intestine \cite{swiss_hospitals_pathogens}. If present in other parts of the body \textit{E. coli} can cause infections \cite{swiss_hospitals_pathogens}. As shown in  Figure \ref{figure:pathogen_dvelopment} infections with resistant ESBL \textit{E. coli} increased rapidly over the last 15 years. 

\subsection{Identification of ESBL \textit{E. coli} and treatment}
The European Committee on Antimicrobial Susceptibility Testing (EUCAST) is an organization advising clinicians on which actions to take in response to results of  antimicrobial susceptibility testing (AST) \cite{leclercq_eucast_2013}. For \textit{E. coli} where clinicians suspect the presence of ESBLs, the EUCAST recommends to confirm ESBLs by a standardized ESBL disc test assay \cite{hombach_consequences_2013}. For this assay, \textit{E. coli} are grown with a fixed density on a plate \cite{disc_test}. On this plate discs with cefotaxime, ceftazidime and cefepime alone, but also in combination with the ESBL inhibitor clavulanic acid, are placed \cite{disc_test}. After an incubation time of 18-24 hours the diameter of the inhibition zone on the plates is measured \cite{disc_test}. If the diameter of the inhibition zones, caused by the antibiotic and clavulanic acid combination, is larger than 5 mm, the tested \textit{E. coli} strain is declared as ESBL producing \cite{disc_test}. In those cases clinicians are recommended to use carbapenems for treatment \cite{mcwilliams_incidence_2014}. This procedure, where ESBL detection is only based on AST was introduced in 2013. Before, the EUCAST advised to detect ESBLs with molecular techniques (\textit{e.g} PCR, microarrays) \cite{screening}. Carbapenems were prescribed, if genes coding for ESBLs were found. Interestingly, many \textit{E. coli} strains carrying ESBL genes were susceptible to common cephalosporins \cite{hombach_consequences_2013}.
Hombach \textit{et al.} showed that 32.5\% of an ESBL \textit{E. coli} isolate collection which all carried the gene coding for CTX-M-1 were susceptible to cefepime \cite{hombach_consequences_2013}. Therefore, carbapenem presicription based on ESBL gene identification was unnecessary in many cases, progressing the carbapenem resistnace evolution. This is why the EUCAST changed the guidelines in 2013 and now only AST is considered. In principal it would be helpful for clinicians to include genomic data again, because AST takes very long (about 48 hours) and is not very accurate \cite{disc_test}\cite{hombach_consequences_2013}.

\section{Identifying mutations predicting cefepime resistance of ESBL \textit{E. coli}}
ESBL genes alone do not determine whether ESBL \textit{E. coli} are resistant to extended-spectrum cephalosporins or not \cite{hombach_consequences_2013}. Instead it is possible that mutations, which have not been identified yet, play an important role in the resistance mechanism. Mutations could affect promoters and genes responsible for various mechanisms. For example mutations in the promoter of ESBL genes could affect the expression level of ESBLs. Enhanced ESBL expression would likely increase hydrolysis of extended-spectrum cephalosporins. Since extended-cephalosporin concentrations in the bacteria is very dependent on transport through porins, it is quite possible that mutations are affecting genes or promoters of porins. Those mutation could affect the expression level of porins but also their strucutre, both leading to increased impermeability. Another possibility for targets of mutations are efflux pumps. Mutated efflux pumps could specifically transport cephalosporins out of the cell, decreasing the cephalosporin concentration in the cell. \\
Mutations can be identified by analyzing genomic data of  ESBL \textit{E. coli} strains gaining resistance by evolution. To get access to such genomic data, the same strain of ESBL \textit{E. coli} can be sequenced before and after resistance advanced. This requires access to a clinical isolate collection where susceptibility changed over time. It is also possible to analyse genomic data obtained from ESBL \textit{E. coli} strains which were experimentally forced to evolve resistance \textit{in vitro}. Studying resistance with experimental evolution is possible with a device called morbidostat.

\subsection{Principles of the morbidostat} 
The morbidostat, originally developed by Toprak \textit{et al.}, is an automated culture device \cite{toprak_building_2013}. It forces bacteria to gain resistance by applying a constant high antibiotic pressure. Bacteria are grown in a fixed culture volume in vials. The growth in every vial is constantly monitored. Depending on how fast the bacteria grow an appropriate dose of antibiotics is injected into the vials. Culturing multiple days leads to evolution of resistance as phenotypes with increased resistance are constantly selected. \\
Several tasks are grouped in cycles and repetitively executed by the morbidostat. Over the defined cycle time the optical densities (ODs) of the cultures are constantly measured which is represented as red dots in Figure \ref{figure:principle}. At the end of the cycle a fit is calculated approximating all the OD measurements from one cycle which is shown in Figure \ref{figure:principle} as a black line going through the red dots. From that fit the growth rate is calculated. A feedback algorithm decides if and how much drug is injected in which vial. The calculated antibiotic concentration is injected with computer-controllable pumps. When liquids are injected, the cultures are diluted as well, which is visible in the first OD measurement of the next cycle. As a last step of the cycle, volume which exceeds the culture volume is removed. As resistance emerges, samples of the cultures grown in the morbidostat can be taken and sequenced.    
\begin{figure}
	\includegraphics[width=0.5\textwidth]{growth_diagram.png}
	\caption{Expected outcome for three cycles of culturing with the morbidostat. Individual OD measurements of a cycle are shown as red dots. A fit is calculated approximating every OD measurement. This fit is used for calculating growth rates determining over injected dose of antibiotics. Antibiotics are injected at the end of a cycle, next to inhibition also causing dilution which is visible in the OD measurements of the next cycle.}
	\label{figure:principle}
\end{figure} 

\subsection{Previous morbidostat experiments and expected outcome}
Dösselmann \textit{et al.} rebuild the system in 2015  \cite{doselmann_rapid_2017} and used it for studying mutations associated with colistin resistance. They were able to increase the minimal inhibitory concentration (MIC) of colistin against \textit{Pseudomonas aeruginosa} 100-fold within 20 days with the morbidostat \cite{doselmann_rapid_2017}. The mechanism of action of colistin is to displace cations from the phospate groups of membrane lipids. This leads to the disruption of the outer cell membrane causing cell death \cite{noauthor_colistin:_nodate}. Dösselmann \textit{et al.} could identify several mutations in genes coding for proteins involved in the lipopolysaccharide synthesis \cite{doselmann_rapid_2017}.

\subsection{Illumina and Nanopore sequencing}
Studying changes in the genome relies on accurate sequencing data. Illumina sequencing is a method that produces short reads which are typically 150 base pairs (bp) long. The advantage of Illumina sequencing is the low error rate which was determined as 0.24 \% per base \cite{pfeiffer_systematic_2018}. A commonly used Illumina sequencing system is the MiSeq-System. As the reads are only 150 bases long, it is computationally difficult to assemble whole-genomes because overlapping regions are small.  
Oxford Nanopore Technologies (ONT) produces very long reads which are up to several 100 kbp long \cite{noauthor_resolving_nodate}. In contrast to Illumina sequencing, the error rate per base is 13.6 \% which is a lot higher compared to Illumina sequencing \cite{noauthor_resolving_nodate}. Most of those errors are caused by repetitive regions in the genome. ONT has issues in detecting those regions correctly \cite{noauthor_resolving_nodate}. However, the detection of single nucleotide polymorphisms (SNPs) has a smaller error rate, as false nucleotide substitutions are found only at a rate of 3\%. False nucleotide substitutions are only found at a rate of 3\% \cite{noauthor_resolving_nodate}. The long reads allow to assemble structurally correct whole-genomes. \\
It is possible to combine the benefits of the long reads of Nanopore with the low error rate of Illumina in a hybrid-assembly. Hybrid-assembling uses the long reads from ONT for scaffolding and the Illumina reads for single-nucleotide accuracy. This results in very accurate whole genome assemblies \cite{noauthor_resolving_nodate}.

\chapter{Aim of this thesis}
The aim of this master thesis was to investigate the resistance evolution of ESBL \textit{E. coli} against the extended-spectrum cephalosporin cefepime. The focus was on identifying  SNPs in the genome of resistant ESBL \textit{E. coli}. One part of achieving this aim was to study genomic data of ESBL \textit{E. coli} isolated from patients, which showed a change in susceptibility against cefepime over time. The other part involved assembling a morbidostat to force ESBL \textit{E. coli} to gain resistance to cefepime and to follow the evolution of resistance by studying genomic data. The specific aims were:
\begin{itemize}
	\item DNA sequencing with Oxford Nanopore Technologies
	\item De novo assembling of ESBL \textit{E. coli} whole-genomes combining Illumina and Nanopore Technology sequencing data
	\item Development of a bioinformatic analysis pipeline to identify SNPs as a product of resistance evolution
	\item Assembling and troubleshooting of the morbidostat, use it to experimentally evolve resistance to cefepime with ESBL \textit{E. coli} and cefepime
	 
	\item Identification of SNPs in resistant ESBL \textit{E. coli} patient isolates
	\item Identification of SNPs in resistant ESBL \textit{E. coli} where resistance evolved experimentally with the morbidostat
\end{itemize}  