\documentclass[12pt,twoside]{report}
%load packages
\usepackage[utf8]{inputenc}
\DeclareUnicodeCharacter{2009}{\,} 
\usepackage[english]{babel}
\usepackage{csquotes}
\usepackage[a4paper,width=150mm,bindingoffset=6mm,top=25mm,bottom=25mm,headheight=15pt]{geometry}
\usepackage{graphicx}
\usepackage{float}
\graphicspath{{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Main/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Introduction/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Materialsandmethods/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Results/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Materialsandmethods/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Supplementary/Images/}{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Declaration/Images/}}
\usepackage{amsmath}
\usepackage{refcheck}
\usepackage{textgreek}
\usepackage{hyperref}
\usepackage{gensymb}
\usepackage{lmodern,textcomp}
\usepackage{color, colortbl}
\definecolor{LightCyan}{rgb}{0.88,1,1}
\usepackage{texshade}
\usepackage{longtable}
\usepackage{dnaseq}
\usepackage{tabularx}
\usepackage{pdfpages}
\usepackage{multirow}
	\newcolumntype{L}{>{\raggedright\arraybackslash}X}
\usepackage[table,xcdraw]{xcolor}
\usepackage{dirtytalk}


\bibliographystyle{unsrt}

\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhead{}
\fancyhead[LO,LE]{\leftmark}
%\fancyhead[RO,RE]{\rightmark}
\fancyfoot{}
\fancyfoot[LO,LE]{Eric Ulrich}
\fancyfoot[CO,CE]{\thepage}
\fancyfoot[RO,RE]{University of Basel}




\usepackage[backend=biber,sorting=none,style=numeric,citestyle=numeric]{biblatex}
\usepackage{url}
\setcounter{biburllcpenalty}{7000}
\setcounter{biburlucpenalty}{8000}
\addbibresource{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Main/bibliography.bib}
\begin{document}

\begin{titlepage}
	\begin{center}
		\includegraphics[width=0.4\textwidth]{logo}\\
		\vspace{1cm}
		\Large
		Faculty of Science \\
		Department Biozentrum\\
		University of Basel\\
		Switzerland\\
		\date{\today}
		\vspace*{1cm}
		
		\Huge
		Master Thesis Molecular Biology \\
		\vspace{2cm}
		\Huge
		\textbf{Experimental evolution and genotypic characterization of ESBL \textit{Escherichia coli}}
		
		\vspace{3cm}
		\begin{minipage}[t]{0.47\textwidth}
			\textnormal{\large{\bf Submitted to:\\}}
			{\large Prof. Dr. Richard Neher\\ Prof. Dr. Dirk Bumann}
		\end{minipage}\hfill\begin{minipage}[t]{0.47\textwidth}\raggedleft
			\textnormal{\large{\bf Submitted by:\\}}
			{\large Eric Ulrich}
		\end{minipage}			
		\vspace{3.5cm}
		\newline 
		\large{\today}

		
		
	\end{center}
\end{titlepage}
\pagenumbering{roman}
\chapter*{Abstract}
%\addcontentsline{toc}{chapter}{Abstract}
In Swiss hospitals infections with extended-spectrum cephalosporin resistant \textit{Escherichia coli} (\textit{E. coli}) are emerging \cite{swiss_hospitals}. Enzymes called extended-spectrum \textbeta-lactamases (ESBL) are able to hydrolyze extended-spectrum cephalosporins. If expressed in \textit{E. coli}, the susceptibility to extended-spectrum cephalosporins is reduced and the strains are called ESBL \textit{E. coli}. Research focused on studying ESBL mediated extended-spectrum cephalosporin resistance, even though other resistance mechanisms possibly affect the resistance level. We studied the evolution of cefepime (an extended-spectrum cephalosporin) resistance in ESBL \textit{E. coli} strains in order to identify new genotypes possibly associated with resistance mechanisms. In order to do so, we studied whole-genome sequencing data from ESBL \textit{E. coli} isolates from patients of the University Hospital of Basel. These isolates evolved resistance to cefepime in patients. A bioinformatic pipeline was developed for analyzing the sequencing data, revealing several single nucleotide polymorphisms (SNPs) in genomes of isolates which evolved resistance.
Additionally, we assembled a device called morbidostat, which we used to experimentally evolve resistance to cefepime in ESBL \textit{E. coli} strains. The morbidostat is an automated culturing device continuously applying high antibiotic pressure to bacterial cultures resulting in resistance evolution. We cultured three ESBL \textit{E. coli} strains in the morbidostat with cefepime which highly increased the resistance level of the strains. Samples of strains taken during morbidostat experiments were deep-sequenced and analyzed with the same bioinformatic pipeline which was applied to patient isolates. Compared to the genomes of the strains before culturing with the morbidostat, several SNPs were identified in the genomes of the strains which evolved resistance. Interestingly, we found SNPs in the same genes in patients isolates and morbidostat samples, strongly suggesting the importance of the SNPs for the resistance mechanisms. In particular, we identified SNPs in genes coding for the porins \textit{ompC}, \textit{ompF} and their regulatory system in patient isolates and morbidostat samples. Furthermore, SNPs in genes coding for the DNA-directed RNA polymerase subunit \textbeta \space and the RNA polymerase sigma factor in patient isolates and morbidostat samples affected the transcription machinery.  
\chapter*{Acknowledgments}
%\addcontentsline{toc}{chapter}{Acknowledgments}
I would like to thank Richard Neher and Dirk Bumann who gave me the opportunity to work on this very interesting project. Next, I would like to thank everyone in Neher's group for the stimulating discussions and for the friendly environment. Special thanks go to Beatrice Claudi for producing the ESBL plasmids and for answering every microbiology-related question that I had. Furthermore, I would like to thank Sandra SÃ¶derholm for supporting me so much with this project and for proofreading my thesis. 
Lastly, I would like to thank my family who supported me throughout the whole year and gave me the opportunity to fully focus on this project. 
\tableofcontents
\pagenumbering{arabic}
\chapter{Introduction}

\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Introduction/introduction}

\chapter{Materials and methods}
\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Materialsandmethods/materialsandmethods}

\chapter{Results}
\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Results/results}

\chapter{Discussion}
\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Discussion/discussion}

\chapter{References}
\phantomsection
\let\Origclearpage\clearpage
\let\clearpage\relax
\defbibheading{bibliography}[\refname]{}
\printbibliography
\newpage
\chapter{Supplementary}
\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Supplementary/supplementary}
\input{/home/ericulrich/Documents/Master/Masterarbeit/LaTeX_thesis/Declaration/declaration}
\end{document}
