The discovery of penicillin shortly before the Second World War revolutionized human medicine by saving the lives of millions of soldiers and civilians suffering under bacterial infections \cite{cdc_biggest_2019}. Because bacterial infections could now be treated major advances in surgery were possible \cite{worldwar_resistance}. Since then antibiotics play a very important role in the health care system and we are heavily depending on those drugs.\\
Because of the positive effects of penicillin it quickly started to be used in excess and irresponsible. This led to the emergence of penicillin resistance within 32 years after its discovery \cite{worldwar_resistance}.
Ever since then it was necessary to repetitively introduce new antibiotics to the market because bacteria evolved resistance against previously developed drugs. 

\section{Emergence of antibiotic resistance}
Worldwide an emergence of antibiotic resistance is occurring \cite{ventola_antibiotic_2015}. An increase of antibiotic usage and a decrease of development of new antibiotics are the the main reasons for this emergence \cite{ventola_antibiotic_2015}. 
\begin{figure}
	\includegraphics[width=1\textwidth]{consumption.png}
	\caption{A: Consumption of defined daily antibiotic doses per 1000 inhabitants per day over 15 years.  B: Antibiotic consumption for selected countries associated with certain incomes. Especially India and other countries with a huge population with a lower income show a tremendous increase of antibiotic consumption \cite{klein_global_2018}. C: Decrease of FDA approved drugs between 1980 and 2014.}
	\label{figure:emergence}
\end{figure}
As shown in Figure \ref{figure:emergence} A and B consumption mainly increased for countries where a huge population is associated with middle and lower income \cite{klein_global_2018}. In those two groups the increase between 2000 and 2015 behaved exponentially. Countries where the average income is high show a pretty stable consumption over the recorded years. But the overall consumption is much higher compared to the other groups with middle and low income. This also shows that there is an uneven access to antibiotics. \\
The development of new antibiotics on the other hand decreased in the last four decades as visible in Figure \ref{figure:emergence} C. When between 1980 and 1984 19 antibiotics were approved by the Food and drug  Administration (FDA), only six novel drugs where approved between 2010 and 2014. The decrease of development of new antibiotics mainly has economical reasons. Newly approved antibiotics are usually held in reserve and only prescribed for infections that more established antibiotics can't treat. This heavily limits the investment in return \cite{fair_antibiotics_2014}. Additionally antibiotics are only used for a short time. Compared to drugs used for treating chronic diseases this makes antibiotics a lot less profitable  \cite{fair_antibiotics_2014}. With an estimated cost of 0.5 billion USD development of antibiotics is also very expensive \cite{costs}\\
Antibiotic consumption takes place in two fields. One being the health sector and one being the food industry. Unfortunately in the health sector many antibiotics are prescribed unnecessarily. It was shown that 47 millions prescriptions of antibiotics in the USA were not needed \cite{noauthor_antibiotic_2019}. Most of them were prescribed for respiratory conditions most commonly caused by viruses \cite{noauthor_antibiotic_2019}. In the food industry it is assumed that in the early 2000s 25-50\% of all antibiotic consumption took place \cite{palumbi_humans_2001}. What makes this extremely high usage questionable is that they are manly used for prophylaxis and to stimulate growth but not to cure animals with diseases. \\
The combination of increased usage and decreased development brought up resistance of all major bacterial pathogens. In Figure \ref{figure:pathogen_dvelopment} the most common resistant pathogens in Swiss hospitals are shown. \textit{Staphylococci aurei}, \textit{streptococci pneumoniae}, \textit{snterococcus faecalis} and \textit{enterococcus faecilum} which are resistant against different classes of anitbiotics decreased in their frequency. This is ether because a vaccine was introduced (PCV7 vaccine against several \textit{pneumococcal} strains) or screening of the pathogens was improved. Resistant \textit{escherichia coli} (\textit{E. coli}) are gaining commonness where mainly resistance against the antibiotics fluoroquinolones and extended spectrum \textbeta-lactams occurs. \\
With this thesis I want to focus on extend spectrum \textbeta-lactamase (ESBL) \textit{E. coli}. This pathogen evolved resistance against most cephalosporins wich are antibiotics belonging to the class of \textbeta-lactams.   

\begin{figure}
	\includegraphics[scale=0.2]{pathogens_overview.png}
	\caption{Development of resistant pathogens in Swiss hospitals. \cite{swiss_hospitals_pathogens}}
	\label{figure:pathogen_dvelopment}
\end{figure}

\section{\textbeta-lactams and resistance against this class of antibiotics}
During the 20th century different classes of antibiotics were developed. All of them harm the bacterial pathogen without harming the host because they target specific bacterial metabolic processes. In Figure \ref{figure:antibiotics} the most common classes of antibiotics (black) and their targets (red) are shown.
\begin{figure}
	\includegraphics[width=0.8\textwidth]{antibiotics.jpg}
	\caption{Most popular classes of antibiotics and their targets in the bacterial cell \cite{wright_english:_2010}}
	\label{figure:antibiotics}
\end{figure}
\textbeta-lactams are by far the most commonly prescribed class of antibiotics.

\subsection{\textbeta-lactams}

\subsubsection{Mechanism of action}
\textbeta-lactam antibiotics act by inhibiting the synthesis of the peptidoglycan layer of the bacterial cell wall \cite{noauthor_-lactam_2019}. A damaged cell wall leads to bursting of the cell caused by osmotic pressure. Gram-positive bacteria are especially vulnerable to \textbeta-lactams because the peptidoglycan forms the very outer layer of their cell membrane \cite{graevemoore_english:_2008}.\\
Peptidoglycan synthesis is inhibited becuase \textbeta-lactams are analogues of the amino acid D-alanyl-D-alanine \cite{fisher_bacterial_2005}. Those tow amino acids form the terminal residues of the peptide which is attached to N-acetylmuramic accid (NAM). NAM itself is linked repetitively with N-acetylglucosamine (NAG) \cite{fisher_bacterial_2005}. The peptide residues of the poly-NAM-NAG chains can be crosslinked by DD-transpeptidases. This cross-linkage results in a mesh-like layer the peptidogylcan \cite{fisher_bacterial_2005}. \textbeta-lactams bind to the active site of DD-transpeptidases and inhibit their activity irreversibly \cite{fisher_bacterial_2005}.

\subsubsection{Cephalosporins}
The \textbeta-lactams contain a highly strained and reactive cyclic amide. Five groups of \textbeta-lactams exist. Those being penams, penems, carbapenems, monobactams and the cefems \cite{beta-lactam_nodate}. 
Cephalosporins belonging to the cephems are very important because they showed to be very potent and well tolerated by patients thus they are widely used antibiotics \cite{dancer_problem_2001}. Cephalosporins are divided into five generations.\\
\textbf{First-generation cephalosporins} are very active against gram-positive cocci but are not very potent against gram-neagtive bacteria \cite{fernandes_-lactams:_2013}. \textbf{The second generation of cephalosporins} are all active against bacteria covered by first-generation drugs, but have extended coverage against gram-negative bacteria \cite{fernandes_-lactams:_2013}. The tendency of enhanced activity against gram-negative bacteria is further visible in \textbf{third-generation cephalosporins}. A common representative of this generation is ceftazidime \cite{klein_third-generation_1995}. The Second and third generation of cephalosporins are also known as extended cephalosporins. Only two beta-lactams are classified under the \textbf{fourth-generation of cephalosporins}, those being cefepime and cefpirome \cite{fernandes_-lactams:_2013}. The newest \textbf{fifth generation of caphalosporins} were explicitly developed to target resistant strains of bacteria but unfortunately drugs belonging to this generation are ineffective against enterococci bacteria \cite{fernandes_-lactams:_2013}.\\

\subsection{Resistance of ESBL \textit{E. coli} against \textbeta-lactams}
\label{section:resistance_mechanisms}
It is assumed that ESBL \textit{E. coli} gained resistance by expressing enzymes called \textbeta-lactamases which are albe to hydrolyze \textbeta-lactams \cite{fisher_bacterial_2005}.
\textbeta-lactamases are able to hydrolyze the cyclic amid of \textbeta-lactams \cite{noauthor_beta-lactam_nodate}. Hydrolyzed \textbeta-lactam is no longer able to inhibit the DD-transpeptidase \cite{noauthor_beta-lactam_nodate}.

\subsubsection{\textbeta-lactamases}
Mainly plasmid-mediated \textbeta-lactamases are important for resistance because they can be transmitted via horizontal gene-transfer \cite{munita_mechanisms_2016}.  
The first plasmid-mediated \textbeta-lactamase in gram-negative bacteria was described in the early 1960s and called \textbeta-lactamase TEM-1 \cite{fernandes_-lactams:_2013}. TEM-1 was found worldwide only few years later after its first isolation \cite{fernandes_-lactams:_2013}. TEM-1 was able to cause resistance against \textbeta-lactams of this time.  That is why extended cephalosporins such as ceftazidime were developed which couldn't be hydrolyzed by TEM-1 \cite{fernandes_-lactams:_2013}. \\
Another anciently known \textbeta-lactamase is called SHV-1 which was isolated for the first time in 1974 \cite{kuzin_structure_1999}. It is encoded chromosomally in the majority of isolates of \textit{K. pneumoniae} but is also plasmid mediated when present in \textit{E. coli} \cite{kuzin_structure_1999}. \\
\textbeta-lactamases which are able to hydrolyze extended-spectrum cephalosporins are called extended-spectrum \textbeta-lactamases or ESBLs. This resulted in the name ESBL \textit{E. coli} for  \textit{E. coli} expressing such \textbeta-lactamases. 

\subsubsection{ESBLs }
Next to the TEM and SHV families new \textbeta-lactamases were isolated over. One of them is called CTX-M-1 \textbeta-lactamase which was clinically isolated for the first time in Germany in 1986 \cite{bradford_extended-spectrum_2001}. Because it only has a sequence identity of 40 \% compared to TEM or SHV \cite{bradford_extended-spectrum_2001} it was categorized as a new \textbeta-lactamase family. Later on many variants were isolated and the CTX protein family was subdivided into five subgroups, with CTX-M-1 being one of them  \cite{fernandes_-lactams:_2013}.
It is assumend that they evolved from the \textbeta-lactamase precursor AmpC from \textit{Klyzvera ascorbata}  \cite{bradford_extended-spectrum_2001}. Even though the first CTX-M-1 was isolated in Germany, it is mostly popular in eastern Europe, South America and Japan \cite{bradford_extended-spectrum_2001}. \\
Another \textbeta-lactamase family belonging to the ESBLs is called OXA.
The OXA family was originally created as a phenotypic rather than a genotypic group, based on a specific hydrolysis profile \cite{bradford_extended-spectrum_2001}. Its name comes from the ability to efficiently hydrolyze oxacillin \cite{bradford_extended-spectrum_2001}. \\
Originally susceptible to extended cephalosporins, some TEM-1 variants also evolved resistance against extended cephaloporins. 

\subsection{General resistance mechanisms}
Generally there are three different strategies for a bacterial cell to reduce susceptibility against antibiotics. One strategy is to make it more difficult for the antibiotic to enter the cell which is effective because many antibiotics have target sites within the cell \cite{barreteau_cytoplasmic_2008}. The second strategy is to reduce the affinity or to protect the target site of the antibiotic. The last strategy is to  to reduce the amount of antibiotic molecules which are already within the cell. This is possible by degradation of the compound which we have seen with \textbeta-lactamases or by active transport out of the cell.\\
Some compounds such as \textbeta-lactams rely on channels in the membrane of the bacteria called porins in order to reach their target site within the cell. That is because e.g. \textbeta-lactams are hyrdophylic and therefor can not just pass the lipophylic membrane by diffusion \cite{munita_mechanisms_2016}. Because antibiotics have very specific target sites another mechanism to gain resistance is to change the structure of the target \cite{munita_mechanisms_2016}. Some pathogens also protect their target sites by dislodging the compound by increasing the dissociation constant \cite{connell_ribosomal_2003}.
Active transport of antibiotics out of the cell is possible because bacterial cells are able to express substrate specific efflux pumps. Following this pathway is a widely spread mechanism against most classes of antibiotics \cite{munita_mechanisms_2016}.

\subsection{ESBL \textit{E. coli}}
In order to minimize unnecessary prescriptions of last resort antibiotics EUCAST publishes breakpoints for every antibiotic. Those breakpoints advice clinicians after which minimal inhibitory concentrtaion (MIC) a pathogen is seen as resistant. They also publish guidelines which should help in choosing the appropriate antibiotic. Interestingly presence of ESBL genes does not guarantee resistance against extended cephalosporins. This means that some ESBL \textit{E. coli} have MICs below the breakpoint published by the EUCAST \cite{tissot_enterobacteriaceae_nodate}. It is very important that in such cases cephalosporins are prescribed instead of carbapenems or other last resort antibiotics which are used to treat resistant ESBL \textit{E. coli}. \\
In the past when ESBL genes were detected by molecular screening the EUCAST recommended to prescribe last resort antibiotics no mather if MIC determination showed cephalosporin susceptibility \cite{leclercq_eucast_2013}. By now the EUCAST changed their guidelines and only susceptibility testing determines which antibiotic is chosen for treatment \cite{leclercq_eucast_2013}. Unfortunately MIC determination takes about 48 hours which is too long in some cases forcing prescription of carbapenemens even though it is unknown if it is actually necessary.\\
It is not known what the difference is between ESBL \textit{E. coli} which are susceptible or resistant against cephalosporins.   

\section{Identifying mutations deciding over resistance or suscpetibility of ESBL \textit{E. coli}}
Our goal is to identify mutations which potentially explain if an ESBL \textit{E. coli} strain is resistant or susceptible to extended cephalosporins. Since ESBL genes alone don't determine the resistance we assume that additional mutations are needed in order that ESBL \textit{E. coli} show resistance. Regulatory sequences for example of the ESBL genes could be affected by such mutations causing up-regulation of ESBLs. Other options would be that genes involved in a different resistance mechanism than \textbeta-lactam hydrolysis are affected by mutations. Those could be genes coding e.g. for porins, efflux pumps or DD-transpeptidases. Of course it is also possible that it is a combination of mutations triggering differenet resistance mechanisms simultaneously. \\
In order to identify such mutations we pursue two different approaches. First we want to identify mutations which accumulated in clinical isolates of ESBL \textit{E. coli} which gained resistance over time. This is possible because the Univeristy hopital of Basel established a sample collection of ESBL \textit{E. coli}. \\
For the other approach we want to establish a morbidostat which is a device which automatically allows to force bacteria to evolve resistance. We want to use the morbidostat to force \textit{E. coli} carrying a plasmid coding for an ESBL to evolve resistance against cefepim a fourth-generation cephalosporin. Similar to the first approach we want to identify mutations which accumulated over the process of resistance evolution.

\subsection{Principles and experiences with the morbidostat} 
The morbidostat was initially invented by Toprak et al \cite{morb_toprak}. Later on Neher et al \cite{doselmann_rapid_2017} rebuilt the system in 2015. Based on those two systems we rebuild a morbidostat with small modifications which should improve the handling and mainly make it a lot smaller in size. The principle is, that cultures are grown in vials. The growth in every vial is monitored and depending on how fast they grow, an appropriate amount of antibiotics is pumped into the vials. Therefore the cultures are constantly inhibited with antibiotics. This means that high antibiotic pressure is applied to the cultures. By culturing for multiple days this leads to evolution of resistance based on selection for phenotypes with lower suceptibility to the antibiotic.
In the left Figure \ref{figure:vial_setup} the principle of the morbidostat is shown. Over a defined cycle time the optical densities (ODs) are recorded. Measuring the OD works by sending a beam of light through the cultures with an LED. The cells cause scattering of the light beam which is detected by photo-transistors. After the end of a cycle which is typically 10 minutes the growth during this period is calculated. Based on that growth a feedback algorithm decides how much antibiotics is injected into the vials. Because different concentrations need to be injected, each vial is conncected to three bottles with different concentrations ranging from zero to highly concentrated. The calculated concentration is injected by turning on the pumps conncected to the different concentrations for certain times. To avoid overflowing volume exceeding a defined culture volume needs to be removed by another pump. \\
In the right Figure \ref{figure:vial_setup} the expected ODs of multiple cycles is shown. The red dots represent the OD measurements which were collected over once cycle. The line which is the fit through the red points is the calculated growth. At the end of the cycle antibiotics are injected which causes next to inhibition also dilution. That is why the first OD measurement of the next cycle is lower compared to the previous cycle.
With our morbidostat we are able to culture 15 bacterial strains independently. Because we store how much antibiotics are injected into each vial we are also able to judge how far evolution of resistance progresses. We want to culture \textit{E. coli} carrying a plasmid with ESBL genes with our morbidostat. We produce those strains by cloning plasmids carrying \textbeta-lactamase sequnces into K-12 \textit{E. coli}. By culturing those strains with the morbidostat using cefepim as antibiotic we want to force evolution of resistance against this drug. During this process we want to take samples. We analyze those samples using next-generation sequencing technologies. Because we work with cloned strains we need to de-novo assemble the genome of those produced strains. We do that by combining Nanopore whole-genome sequencing data with Illumina data. Samples that we collect during morbidostat experiments we deep sequenced with Illumina sequencing.
\begin{figure}
	\includegraphics[width=0.5\textwidth]{vial_diagram.png}
	\includegraphics[width=0.5\textwidth]{growth_diagram.png}
	\caption{Left: Principle of the morbidostat. Control flow is organized in cycles starting from measuring growth, ending in injecting an antibiotic concentration calculated by a feedback algorith. Right: Expected outcome of three morbidostat cycles. Red dots represent OD measurements, fit growth is the calculated growth. Injection after each cycle also leads to dilution which is visible at the beginning of the next cycle. \textDelta OD represents the growth despite inhibition and dilution.}
	\label{figure:vial_setup}
\end{figure} 

\subsection{Illumina and Nanopore sequning}
Illumina sequencing is a method which produces short reads which are typically 150 base pairs (bp) long. The strong-suit of Illumina sequecning is the low error rate which was dertermined as 0.24 \% per base \cite{pfeiffer_systematic_2018}. A common used Illumina sequecing system is the MiSeq-System which is also used for this project. Because the reads are only 150 bases long it is computationally not possible to assemble structurally correct whole-genomes. 

Oxford Nanopore Technologies produces very long reads which are up to several 100 kbp long. In contrast to Illumina sequencing the error rate per base is 13.6 \% \cite{noauthor_resolving_nodate} which is a lot higher. But because the reads are so long it is possible to assemble structurally correct whole-genomes. \\
By sequencing on both platforms it is possible to combine the benefits of the long reads of Nanopore with the low error rate of Illumina. This results in very accurate whole genome assemblies.

\subsection{Previous morbidostat experiments and expected outcome}
Dösselmann et al \cite{doselmann_rapid_2017} used the morbidostat in order to study colistin resistance. With the morbidostat they were able to increase the MIC of colistin for Pseudomanas aeruginosa a 100-fold within 20 days \cite{doselmann_rapid_2017}. They identified a mutation pattern associated with colisitn resistance. The mechanism of action for colistin is to displace cations from the phospate groups of membrane lipids. This leads to disruption of the outer cell membrane causing cell death \cite{noauthor_colistin:_nodate}. Therefor it is not surprise that Dösselmann et al could identify several mutations coding fro proteins involved in the lipopolysaccharide synthesis \cite{doselmann_rapid_2017}.