With this thesis I studied the evolution of cephalosporin resistance of ESBL \textit{E. coli}. I established a bioinformatic pipeline which we used to identify changes in the genome while further resistance to cephalosporines evolved in ESBL \textit{E. coli}. This pipeline was applied to next generation sequencing data coming from ESBL samples isolated from patients at the University Hospital of Basel. Furthermore, I assembled a morbiostat which we used to experimentally evolve resistance with ESBL \textit{E. coli} to cefepime. Samples taken from the morbidostat were analyzed with the established bioinformatic pipeline. 

\section{Technical and experimental challenges with the morbidostat}
\subsection{Hardware issues}
We faced problems with the computer controllable pumps that we used for injecting antibiotics and media. The controllers and the piezo pumps reacted very sensitive to small currents flowing through the digital pins of the microcontroller. This caused malfunctions of the pumps. We were able to fix this issue by connecting pull-down resistors the digital pins of the microcontroller and the ground. Furthermore, we connected inverters in serial to the digital pins which increased the reliability of the pumps. We saw in the morbidostat experiment 01, that stirring created air cones which affected th OD measurements. This is why we had to reduce the stirring to a minimum. Generally strong stirring would be beneficial because it would guarantee equal antibiotic concentrations in the vials. We could increase stirring if we would lower the OD measuring units. For controlling the morbidostat we used a 2560 Arduino mega microcontroller which has a total of 54 digital pins. In fact we used 53 digital pins which means that we were very close to the limit. If there would be the need of adding computer controllable elements to the morbidostat, another microcontroller would have to be installed. 

\subsection{Software issues}
Our microcontroller could only execute one task after another. This means that we had to ensure, that the python scripts were sending the commands one after another, instead of sending multiple commands at once. Therefore, we had to thread the command.  Even though threaded the commands we encountered a problem with the microncontroller once caused by receiving multiple commands at once. Potentially the OD measurements interfered with changing the state of the digital pins and the microcntroller crashed. All the pumps did not turn off and all the vials were overflowing. Therefore, we added a reset function which recognizes when the microcontroller crashes and automatically hard resets the microcontroller. 

\subsection{\textit{Bacillus cereus} contamination of the morbidostat experiments}
The majority of our vials were contaminated with \textit{Bacillus cereus}. Since all the stocked strains which we used for starting morbidostat experiments were not contaminated, the contamination entered the system during the experiment, or sterilization was not successful. For sterilizing we used 1 L of 3\% citric acid and 1 L of 3\% bleach. Both solutions were pumped through all the tubing over one hour. Additionally every piece of hardware, except the pumps, were autoclaved. \textit{Bacillus cereus} is a gram-positive, endospores forming bacteria \cite{bintsis_foodborne_2017}. It is possible that \textit{Bacillus cereus} endospores overcame the sterilization protocol. A study reported that after exposing \textit{Bacillus cereus} endospores with 10\% bleach for 15 minutes they were still able to produce colonies \cite{robertson_effect_2018}. Alternatively the \textit{Bacillus cereus} contamination entered the system while taking samples or exchanging the media and drug bottles. \\
Cefepime is supposed to be especially active against gram-positive bacteria because the peptidoglycan is not protected by an outer membrane \cite{sykes_chapter_2014}. Therefore, it is generally surprising, that \textit{Bacillus cereus} survived culturing with high doses of cefepime. Possibly \textit{Bacillus cereus} was able to up take the ESBL plasmid from the ESBL \textit{E. coli} strains by natural competence, which decreased the susceptibility. Alternatively \textit{Bacillus cereus} was killed by the high cefepime concentration but the high antibiotic pressure induced the formation of endospores. For sequencing we inoculated LB with kanamycin with the sample stocks and cultured them overnight. During this steps the endospores could have switched to the vegetative cycle causing the contamination in the stocks passed for sequencing.

\section{Bioinformatic challenges}
We only analyzed isolates of a patient with our bioinformatic pipeline where we ensured with phylogenetic analysis that all the samples were the same ESBL \textit{E. coli} strain. Isolates of patient 16 revealed 21 SNPs. It is unlikely that all of those SNPs are a result of resistance evolution. It is possible that the patient got reinfected with a slightly different ESBL \textit{E. coli} strain which resulted in this high number of SNPs in the isolates. Generally it was not possible to ensure that identified SNPs are related to resistance evolution. Likely many patients were treated with other drugs than antibiotics as well which could also cause evolution of the ESBL \textit{E. coli} strains. \\
The weakness of the bioinformatic pipeline was the annotation of the SNPs. For some SNPs no annotation was found by prokka \cite{seemann_prokka:_2014}, the tool we used for annotating our reference genome, but blasting of the affected sequences revealed annotation for some SNPs with no annotation.
For consistency we only presented annotation found by prokka \cite{seemann_prokka:_2014}. 

\section{Biological importance of operons targeted by mutations}
The following section describes operons targeted by SNPs in patient isolates and morbidostat samples where literature was available describing their role in the resistance to cephalosporins, or where the operons are part of a pathway possibly important for cephalosporin resistance.

\subsubsection{OrtT}
OrT is a toxin causing membrane damage resulting in reduced growth and increased persistence during stress, related to amino acid and DNA synthesis \cite{islam_orphan_2015}. Cells in the persistent state are more tolerant to antibiotics \cite{islam_orphan_2015}. Treating cells with several antibiotics such as sulfamethoxazole increased the expression of OrtT 80 fold. 

\subsection{DNA-directed RNA polymerase subunit \textbeta}
The DNA-directed RNA polymerase subunit \textbeta \space is encoded by the gene \textit{rpoB}. One paper suggests the importance  of \textit{rpoB} mutations for \textbeta-lactam resistance. Samantha Palace \textit{et al.} studied three ceftriaxone (a third-generation cephalosporin) resistant \textit{Neisseria gonorrhoeae} isolates and found one mutation affecting the \textit{rpoB} gene in all isolates \cite{palace_rna_2019}. The mutation reported by Samantha Palca \textit{et al.} was affecting a different position in the \textit{rpoB} gene than we found in the isolate of patient 16. By introducing the mutation with PCR in ceftriaxone susceptible \textit{Neisseria gonorrhoeae} strains they were able to achieve similar ceftriaxone MICs to the resistant isolates, indicating the importance of the mutated \textit{rpoB} gene for the resistance mechanism \cite{palace_rna_2019}. It is not understood yet, how the mutated \textit{rpoB} leads to ceftriaxone resistance. \\


\subsubsection{RNA polymerase sigma factor RpoD}
RpoD is a sigma factor which targets RNA polymerase to a wide range of promoters \cite{maciag_vitro_2011}. The same paper which found mutated \textit{rpoB} genes in ceftriaxone resistant \textit{Neisseria gonorrhoeae} isolates, also found mutated \textit{rpoD} genes in some isolates. The mutation that we found affects a different postion than the one found by Samantha Palace \textit{et al.}. They were able to reproduce resistance by mutating the \textit{rpoD} gene of a ceftriaxone suscecptilbe \textit{Neisseria gonorrhoeae} strain. Similar to mutations in the \textit{rpoB} gene, it is not understood yet, how mutations in the \textit{rpoD} gene can lead to resistance.  

\subsection{FepA promoter}
One mutations was found in the promoter regulating the expression of the protein FepA. FepA is a gated outer membrane porin transporting unspecific hydrophilic substances \cite{liu_permeability_1993}. Potentially the mutation in the promoter of FepA reduced the expression which potentially reduces the uptake of cephalosporins.  

\subsection{Transcriptional regulatory protein OmpR}
OmpR is a DNA-binding protein that interacts with the upstream promoter region of the \textit{ompF} and \textit{ompC} genes and regulates their expression \cite{rampersaud_ompr_1994}. OmpC and OmpF are the two major outer membrane proteins in \textit{E. coli} and have been reported for playing an important role in \textbeta-lactam resistance \cite{rampersaud_ompr_1994} \cite{fernandez_adaptive_2012}.

\subsubsection{Osmolarity sensor protein EnvZ}
As OmpR, EnvZ is involved in regulating the expressions of OmpC and OmpF \cite{rampersaud_ompr_1994}. EnvZ is a transmembrane sensor which detects the osmotic pressure and phosporolates OmpR depending on the detected osmolarity \cite{rampersaud_ompr_1994}. At low osmolarity, EnvZ modulates low levels of phosphorylated OmpR, at high osmolarity OmpR mediates high levels of phosphorylated OmpR \cite{rampersaud_ompr_1994}. Phosphorylated OmpR increases the expression of \textit{ompC} gene while repressing the expression of \textit{ompF} gene \cite{rampersaud_ompr_1994}. If OmpR is not phosphorylated, expression of the gene \textit{ompF} is increased and \textit{ompC} repressed. Becerio A \textit{et. al} reported that two cefepime resistant ESBL \textit{E. coli} isolates lack the OmpC and OmpF porins \cite{beceiro_false_2011}. A lack of those porins likely heavily reduces the cefepime uptake.   

\subsection{Outer membrane protein F OmpF}
Cephalosporin transport occurs partly through the outer membrane protein F \cite{masi_structure_2013}. A mutation in the \textit{ompF} gene could affect the porin structure which potentially affects the cephalosporin transport. 

\subsubsection{\textbeta-lactamase CTX-M-1}
One mutation affected the gene coding for the beta-lactamse CTX-M-1. The resulting variant is not known in the literature so it is not possible to conclude whether its activity is increased or not. 
 

\subsection{Potential resistance mechanism considering mutated operons}
Considering genes targeted by SNPs in patient isolates and morbidostat samples suggests that OmpC and OmpF and their regulatory system are affecting cefepime susceptibility. Both, Ompc and OmpF are highly expressed and important for cefepime transport \cite{rampersaud_ompr_1994} \cite{masi_structure_2013}. Both are regulated by the regulatory protein OmpR. Interestingly a mutated version of OmpR was found in the isolates of patient 16 and in a morbidostat sample. The mutations in the \textit{ompR} gene were not identical, in the patient isolate a missense mutaion was present, in the morbidostat sample a nonsense mutation. OmpR binds to the operons of \textit{ompC} and \textit{ompF} which is necessary for their expression \cite{masi_structure_2013}. If OmpR is mutated it can possibly not bind to the operons and expression would be highly repressed. This makes sense in particular for the OmpR with the nonsense mutation in the morbidostat sample. The DNA binding activity of OmpR itself is regulated by EnvZ \cite{masi_structure_2013}. EnvZ is a kinase phosphorylating OmpR depending on osmotic pressure. The gene coding for EnvZ was also targeted by a SNP in a morbidostat sample. Possibly this changes the expression level of OmpC and OmpF as well. \\
The SNP altering the \textit{rpoB} could also affect cefepime susceptibility because it was proven to affect ceftriaxone susceptibility. However, it is not understood how a mutation in a RNA polymerase could affect the cephalosporin susceptibility. Interestingly one SNP in the morbidostat samples affected the RNA polymerase sigma factor RpoD. This increases the suspicion that some processes in the transcription affects the cefepime susceptibility, in a way which is not understood yet. \\
In general it seems like resistance to cefepime can be acquired by different pathways. The outer membrane proteins definitely seem to play an important role, as well as RpoD and RpoB. Other options are mutated CTX-M-1 \textbeta-lactamases which we have seen in one sample of the morbidostat, or increased persistance resulting in reduced growth. 