In this thesis I studied the evolution of cefepime resistance of ESBL \textit{E. coli}. I established a bioinformatic pipeline which we used to identify changes in the genome while resistance to cefepime evolved in ESBL \textit{E. coli}. This pipeline was applied to next generation sequencing data coming from ESBL isolates sampled from patients at the University Hospital of Basel. Furthermore, we assembled a morbidostat which we used to experimentally evolve resistance in ESBL \textit{E. coli} to cefepime. Samples taken from the morbidostat were analyzed with the established bioinformatic pipeline. 

\section{Technical and experimental challenges with the morbidostat}
\subsection{Hardware issues}
We faced problems with the computer controllable pumps that we used for injecting antibiotics and media. The controllers and the piezo pumps reacted very sensitive to small currents flowing through the digital pins of the microcontroller. This caused malfunctions of the pumps. We were able to fix this issue by connecting pull-down resistors to the digital pins of the microcontroller and the ground. Furthermore, we connected inverters in serial to the digital pins which increased the reliability of the pumps.\\
We saw in the morbidostat experiment 01, that stirring created air cones which affected th OD measurements. This is why we had to reduce the stirring to a minimum. Generally strong stirring would be beneficial because it would guarantee equal antibiotic concentrations in the vials. Stirring could be increased if the OD measuring units would be lowered.. 
\subsection{Software issues}
Our microcontroller could only execute one task after another. This means that we had to ensure, that the python scripts were sending the commands one after another, instead of sending multiple commands at once. Therefore, we had to thread the commands.  Even though we threaded the commands we encountered a problem with the microncontroller once, caused by receiving multiple commands at once. Potentially the OD measurements interfered with changing the state of the digital pins and the microcntroller crashed. All the pumps did not turn off and all the vials were overflowing. Therefore, we added a reset function which recognized when the microcontroller crashes and automatically hard resets the microcontroller. 

\subsection{\textit{Bacillus cereus} contamination of the morbidostat experiments}
The majority of our vials were contaminated with \textit{Bacillus cereus}. Since all the stocked strains which we used for starting morbidostat experiments were not contaminated, the contamination entered the system during the experiment, or sterilization was not successful. For sterilizing we used 1 L of 3\% citric acid and 1 L of 3\% bleach. Both solutions were pumped through all the tubing over one hour. Additionally every piece of hardware, except the pumps, were autoclaved. \textit{Bacillus cereus} is a gram-positive, endospores forming bacteria \cite{bintsis_foodborne_2017}. It is possible that \textit{Bacillus cereus} endospores overcame the sterilization protocol. A study reported that after exposing \textit{Bacillus cereus} endospores with 10\% bleach for 15 minutes they were still able to produce colonies \cite{robertson_effect_2018}. Alternatively the \textit{Bacillus cereus} contamination entered the system while taking samples or exchanging the media and drug bottles. \\
Cefepime is supposed to be especially active against gram-positive bacteria because the peptidoglycan is not protected by an outer membrane \cite{sykes_chapter_2014}. Therefore, it is generally surprising, that \textit{Bacillus cereus} survived culturing with high doses of cefepime. Possibly \textit{Bacillus cereus} was able to take up the ESBL plasmid from the ESBL \textit{E. coli} strains by natural competence, which decreased the susceptibility. Alternatively \textit{Bacillus cereus} was killed by the high cefepime concentration but the high antibiotic pressure induced the formation of endospores. For sequencing we inoculated LB containing kanamycin with the sample stocks and cultured them overnight. During this steps the endospores could have switched to the vegetative cycle causing the contamination in the stocks to be passed for sequencing.

\section{Bioinformatic challenges}
We only analyzed isolates of a patient with our bioinformatic pipeline where we ensured with phylogenetic analysis that all the samples were the same ESBL \textit{E. coli} strain. Isolates of patient 16 revealed 21 SNPs. It is unlikely that all of those SNPs are a result of resistance evolution. It is possible that the patient got reinfected with a slightly different ESBL \textit{E. coli} strain which resulted in this high number of SNPs in the isolates. Generally it was not possible to ensure that identified SNPs are related to resistance evolution. Likely many patients were treated with other drugs than antibiotics as well which could also cause evolution of the ESBL \textit{E. coli} strains. \\
The weakness of the bioinformatic pipeline was the annotation of the SNPs. For some SNPs no annotation was found by prokka, the tool we used for annotating our reference genome, but blasting of the affected sequences revealed annotations for some SNPs with no annotation \cite{seemann_prokka:_2014}.
For consistency we only presented annotations found by prokka \cite{seemann_prokka:_2014}. 

\section{Potential resistance mechanisms considering mutated operons}
The identified SNPs and their annotation in ESBL \textit{E. coli} patient isolates and morbidostat samples mainly suggested two targets which could play an important role in cefepime resistance. One of the tagets were porins, in particular the outer membrane protein F (OmpF), the outer membrane protein C (OmpC)  and their shared regulatory system. The other target was the RNA polymerase and their sigma factor. 
\subsection{SNPs in porin operons}
In \textit{E. coli} there are two major outer membrane proteins, outer membrane protein F and outer membrane protein C \cite{rampersaud_ompr_1994}. Both outer membrane proteins are porins through which cephalosporins pass the outer membrane \cite{masi_structure_2013}. Depending on the osmolarity of the environment \textit{ompF} or \textit{ompC} is preferably expressed. This is regulated by an osmolarity sensor protein EnvZ. EnvZ is a transmembrane sensor which detects the osmotic pressure and phosphorylates the transcriptional regulatory protein OmpR depending on the detected osmolarity \cite{rampersaud_ompr_1994}. At low osmolarity, EnvZ modulates low levels of phosphorylated OmpR, at high osmolarity EnvZ mediates high levels of phosphorylated OmpR \cite{rampersaud_ompr_1994}. OmpR is a DNA binding protein which binds specific regions depending on its phosphorilation status and is essential for the expression of OmpC and OmpF \cite{rampersaud_ompr_1994}. Phosphorylated OmpR binds to the promoter of OmpC, if OmpR is not phosphorylated it binds to the promoter of OmpF \cite{rampersaud_ompr_1994}. Therefore, phosphorylated OmpR increases the expression of the \textit{ompC} gene while repressing the expression of the \textit{ompF} gene \cite{rampersaud_ompr_1994}. On the other hand, OmpR which is not phsophorylated increases the expression of \textit{ompF} and reduces the expression of \textit{ompR}. A lack of those porins likely heavily reduces the cefepime uptake. \\
We found several SNPs targeting genes of the expressions system of OmpF and OmpC independently in patient isoaltes and morbidostat samples. OneS SNP targeted \textit{envZ} in a patient isolate, \textit{ompR} was the target of a SNP in a patient isolate as well as in a morbiodstat sample. The SNPs in \textit{ompR} caused different mutations in OmpR. In the patient isolate the SNP caused a missense mutation, in the morbidostat samples the SNP caused a nonsense mutation. A mutated OmpR is potentially not able to bind to the the promoters of \textit{ompC} and \textit{ompF} which likely heavily reduced the expression of both outer membrane proteins. This seems especially plausible for the mutated OmpR in the morbidostat sample, because the nonsense mutation heavily impacts the functionality of the protein. The mutated \textit{envZ} found in a patient isolate potentially also affects the expression levles of OmpC and OmpF. \textit{OmpC} and \textit{ompF} itself were also both targets of SNPs in morbidostat samples. While \textit{ompC} was affected by a SNP causing a silent mutation in OmpC, the mutation in \textit{ompF} caused a missense mutation in OmpF. The missense mutation in OmpF could cuase a change of the structure of the porin, which potentially affects the cefepime permeability. Becerio A \textit{et. al} reported that two cefepime resistant ESBL \textit{E. coli} isolates lacked the OmpC and OmpF porins which further increases the importance of those membrane proteins for cefepime resistance \cite{beceiro_false_2011}. \\
We found another SNP in the promoter of \textit{fepA}. FepA is a gated outer membrane porin transporting unspecific hydrophilic substances \cite{liu_permeability_1993}. The mutation in the promoter potentially repressed the expression of \textit{fepA} which may further decrease the membrane permeability for cefepime. 
\subsection{SNPs in \textit{rpoB} and \textit{rpoD}}
We also found that a mutation targeted \textit{rpoB} coding for the DNA-directed RNA polymerase subunit \textbeta \space in a patient isolate. In a morbidostat sample we found a SNP targeting \textit{rpoD} coding for the RNA polymerase sigma factor. This strongly suggests that some process in the RNA transcription is affecting the cefepime susceptibility. In a preprint Samantha Palace \textit{et al.} reported that they found a SNP in \textit{rpoB} in three ceftriaxone ( a third-generation cephalosporin) resistant \textit{Neisseria gonorrhoeae} isolates \cite{palace_rna_2019}. They were able to recreate resistance by integrating the mutation in a ceftriaxone susceptible \textit{Neisseria gonorrhoeae} strain, proving the importance of the mutation for ceftriaxone resistance \cite{palace_rna_2019}. The mutation in \textit{rpoB} that they identified in not the same as the one that we found. Samantha Palace \textit{et al.} also reported a mutation in \textit{rpoD} in two \textit{Neisseria gonorrhoeae} isolates \cite{palace_rna_2019}. They were also able to recreate resistance for this mutation. Interestingly it is not understood yet, how mutations in RpoB or RpoD are impacting cephalosporin resistance. 
\subsection{SNPs affecting other operons}
We also identified SNPs affecting genes where the evidence was less clear that they potentially impact cefepime susceptibility. For example one SNP targeted \textit{ortT}. OrtT is a toxin causing membrane damage resulting in reduced growth and increased persistence during stress, related to amino acid and DNA synthesis \cite{islam_orphan_2015}. Cells in the persistent state are more tolerant to antibiotics \cite{islam_orphan_2015}. Treating cells with several antibiotics such as sulfamethoxazole increased the expression of OrtT 80 fold \cite{islam_orphan_2015}. A mutated OrtT is another candidate which could affect cefepime susceptibility. \\
One SNP affected the ESBL gene coding for \textbeta-lactamase CTX-M-1 in a morbidostat sample. This caused a missense mutation in the ESBL. The resulting variant is not know in the literature, therefore, it is not possible to estimate whether it is potentially hydrolyzing \textbeta-lactams more effectively. \\

To finally conclude if the identified SNPs are affecting cefepime susceptibility, the SNPs would have to be introduced into ESBL \textit{E. coli} strains. Then the MIC of cefepime of the mutated strains could be compared to the wild-type, if it would be significantly increased this would prove the importance of the SNP for the resistance. 