In this thesis the evolution of cefepime resistance in ESBL \textit{E. coli} was studied. A bioinformatic pipeline was established, which we used to identify SNPs in the genome while resistance to cefepime evolved in ESBL \textit{E. coli}. This pipeline was applied to next generation sequencing data of ESBL isolates sampled from patients at the University Hospital of Basel. Furthermore, we assembled a morbidostat which we used to experimentally evolve cefepime resistance in ESBL \textit{E. coli}. Samples taken from the morbidostat were analyzed with the established bioinformatic pipeline as well. 
Technical and experimental challenges are discussed in the following as well as the potential relevance of the identified SNPs and their annotation for cefepime resistance.

\section{Technical and experimental challenges with the morbidostat}
\subsection{Hardware issues}
We faced problems with the computer controllable pumps that we used for injecting antibiotics and media. The controllers and the piezo pumps were very sensitive to small currents flowing through the digital pins of the microcontroller. This caused malfunctions of the pumps. We were able to fix this issue by connecting pull-down resistors to the digital pins of the microcontroller and the ground. Furthermore, we connected inverters in serial to the digital pins, which increased the reliability of the pumps.\\
We observed in the morbidostat experiment 01 that stirring created air cones which affected the OD measurements. This is why we had to reduce the stirring to a minimum. Generally, strong stirring could be beneficial because it would guarantee equal antibiotic concentrations in the vials. Stirring could be increased if the OD measuring units would be lowered. \\
We used vials with a polytetrafluoroethylene (Teflon) insert with holes through which we inserted plastic tubes acting as vial inserts. Even though the holes for the plastic tubes were drilled as small as possible, the plastic tubes started to loosen up over time, which we tried to fix by adding silicone. Because of the physical properties of polytetrafluoroethylene silicone did not stick and the plastic tubes could not be sealed properly. Therefore, some vials may have not been completely sealed, potentially increasing the risk of contamination. Furthermore, the chosen vial design was not ideal for sampling, because we had to unscrew the lid of the vials in the hypoxi-station, which increased the risk of introducing contaminations.
\subsection{Software issues}
Our microcontroller could only execute one task after the other. This means that we had to ensure that the python script was sending the commands one after another, instead of sending multiple commands at once. Therefore, we had to thread the commands.  Even though we threaded the commands, we encountered a problem with the microcontroller, caused by receiving multiple commands at once. The OD measurements potentially interfered with changing the state of the digital pins and the microcontroller crashed. The pumps did not turn off and the vials were overflowing. Therefore, we added a reset function which recognized when the microcontroller crashes and automatically hard resets the microcontroller.   

\subsection{\textit{Bacillus cereus} contamination of the morbidostat experiments}
The majority of our vials were contaminated with \textit{Bacillus cereus}. Since all the stocked strains which we used for starting morbidostat experiments were not contaminated, the contamination must have entered the system during the experiment. Alternatively, sterilization was not successful. For sterilizing we used 1 L of 3\% citric acid and 1 L of 3\% bleach. Both solutions were pumped through all the tubing over one hour. Additionally, every piece of hardware in contact with fluids, except the pumps, were autoclaved. \textit{Bacillus cereus} is a gram-positive, endospore-forming bacteria \cite{bintsis_foodborne_2017}. It is possible that \textit{Bacillus cereus} endospores overcame the sterilization protocol. A study reported that after exposing \textit{Bacillus cereus} endospores to 10\% bleach for 15 minutes, they were still able to produce colonies \cite{robertson_effect_2018}. Alternatively, the \textit{Bacillus cereus} contamination entered the system while taking samples or exchanging the media and drug bottles. As already discussed, the design of the vials was not ideal and contaminations could have entered the device through gaps in the vial inserts. It is also possible that some tube connections were not completely sealed allowing contaminations to enter the device.  \\
Cefepime is supposed to be especially active against gram-positive bacteria \cite{sykes_chapter_2014}. Therefore, it is generally surprising, that \textit{Bacillus cereus} survived culturing with high doses of cefepime. \textit{Bacillus cereus} might have taken up the ESBL plasmid from the ESBL \textit{E. coli} strains by natural competence, which decreased their susceptibility. Possibly, \textit{Bacillus cereus} was killed by the high cefepime concentration, but the antibiotic pressure induced the formation of endospores. For sequencing we inoculated LB containing kanamycin with the sample stocks and cultured them overnight. During overnight culturing the endospores might could have switched to the vegetative cycle, causing the contamination in the stocks to be passed for sequencing.

\section{Bioinformatic challenges}
Generally, we identified more mutations in patient isolates than in morbidostat samples. For example the isolates of patient 16 revealed 21 SNPs. One source for this observation  could be that some patients got reinfected with different ESBL \textit{E. coli} strains, even though we tried to ensure strain identity by phylogenetic analysis. Potentially, the phylogenetic tree was not fully resolved and some strains, assumed to be identical, but were actually different. The environment during morbidostat experiments can be fully controlled, because we cultured in the hypoxi-station. The nutrient concentration remained the same, because we cultured with diluted LB media throughout the whole experiment. Therefore, the antibiotic pressure was the main stress causing evolution. On the other hand, in patient isolates the strains had to adapt to many more environmental factors which could further explain why more SNPs were identified in patient isolates. Likely, many patients were treated with other drugs than antibiotics as well, which could also cause evolution of the ESBL \textit{E. coli} strains. \\
For the isolate series of patient 25 and patient 33 the resistance decreased over the sampling period, which is the opposite compared to the other patient isolate series and the sample series from the morbidostat. For those two resistance did not evolve over the sampling period. Therefore, we defined the most recent sample with the lowest resistance level as reference and identified SNPs in the earliest samples with high resistance levels. We still consider the identified SNPs as relevant, because it is still likely that they can be associated with the corresponding high resistance level. We would have preferred to analyze more isolate series where resistance evolved over the sampling period, but we were restricted by the isolate collection at hand. \\
The bioinformatic pipeline for identifying SNPs proved to be solid. \textit{De novo} whole-genome assembling with ONT and Illumina sequencing data using the assembler Unicycler resulted in accurate whole-genomes, positively affected by the high coverage that we achieved with ONT sequencing \cite{wick_unicycler:_2017}. The rest of the pipeline consisted of established bioinformatic steps such as mapping Illumina reads to reference genomes and identifying SNPs with coverage and base frequency filtering. The weakness of the bioinformatic pipeline was the annotation of the SNPs. For some SNPs no annotation was found by prokka, the tool we used for annotating our reference genome, but blasting of the affected sequences revealed annotations for some additional SNPs \cite{seemann_prokka:_2014}.
However, we only presented annotations found by prokka \cite{seemann_prokka:_2014}.

\section{Culturing with the morbidostat}
The assembled morbidostat and the developed feedback algorithm worked very well. The designed OD measuring units proved to be rather sensitive, with the ability to detect ODs starting from 0.01. After optimizing the control of the injecting pumps, they showed to be very reliable and their compact design allowed easy handling of the morbidostat within the space-limited hypoxi-station.
The feedback reacted very well to the cultures by diluting with media when the bacteria were dying or by increasing antibiotic concentration if they grew very fast. The developed feedback algorithm might have allowed resistance to evolve faster than expected. DÃ¶sselmann \textit{et al.} reported that they cultured \textit{Pseudomonas aeruginosa} with a morbidostat and colistin for 20 days to achieve a 100-fold increase in MIC \cite{doselmann_rapid_2017}. We cultured ESBL \textit{E. coli} for only 5 days with our morbidostat and cefepime to achieve a similar increase of the MIC. Since the organism and the antibiotic are not identical, it is difficult to compare how fast resistance evolved. The performed morbidostat experiments proved that our version of the morbidostat is very capable to evolve resistance over a short period of time. 

\section{Potential resistance mechanisms considering mutated operons}mo
The identified SNPs in ESBL \textit{E. coli} patient isolates and morbidostat samples mainly suggested two targets, which could play an important role in cefepime resistance. One of the targets were porins, in particular the outer membrane protein F (OmpF), the outer membrane protein C (OmpC)  and their shared regulatory system. The other target was the RNA polymerase and the sigma factor. 
\subsection{SNPs in porin operons}
There are two major outer membrane proteins \textit{E. coli}, OmpF and OmpC \cite{rampersaud_ompr_1994}. Both are porins in the outer membrane through which cephalosporins pass \cite{masi_structure_2013}. A lack of those porins likely heavily reduces the cefepime uptake. Becerio A \textit{et. al} reported that two cefepime resistant ESBL \textit{E. coli} isolates lacked the OmpC and OmpF porins showing the importance of those porins for cefepime resistance \cite{beceiro_false_2011}.  Depending on the osmolarity of the environment \textit{ompF} or \textit{ompC} is preferably expressed. This is regulated by an osmolarity sensor protein EnvZ. EnvZ is a transmembrane sensor which detects the osmotic pressure and phosphorylates the transcriptional regulatory protein OmpR depending on the detected osmolarity \cite{rampersaud_ompr_1994}. At low osmolarity, EnvZ modulates low levels of phosphorylated OmpR, at high osmolarity EnvZ mediates high levels of phosphorylated OmpR \cite{rampersaud_ompr_1994}. OmpR is a DNA-binding protein which binds specific regions depending on its phosphorylation status and it is essential for the expression of \textit{ompC} and \textit{ompF} \cite{rampersaud_ompr_1994}. Phosphorylated OmpR binds to the promoter of \textit{ompC}, if OmpR is unphosphorylated it binds to the promoter of \textit{ompF} \cite{rampersaud_ompr_1994}. Therefore, phosphorylated OmpR increases the expression of the \textit{ompC} gene while repressing the expression of the \textit{ompF} gene \cite{rampersaud_ompr_1994}. On the other hand, OmpR which is not phosphorylated increases the expression of \textit{ompF} and reduces the expression of \textit{ompC}. \\
We found several SNPs in genes of the expressions system of \textit{ompF} and \textit{ompC} in patient isolates and morbidostat samples. One SNP in \textit{envZ} in a patient isolate, in \textit{ompR} one SNP was found in a patient isolate as well as in a morbidostat sample. In the patient isolate the SNP caused a missense mutation in OmpR, whereas in the morbidostat samples the SNP caused a nonsense mutation in OmpR. A mutated OmpR is potentially not able to bind to the the promoters of \textit{ompC} and \textit{ompF}, which could heavily reduce the expression of both outer membrane proteins. This seems especially plausible for the mutated OmpR in the morbidostat sample, because the nonsense mutation heavily impacts the functionality of the protein. The mutated \textit{envZ} found in a patient isolate potentially also affects the expression levels of \textit{ompC} and \textit{ompF}. In \textit{ompC} and \textit{ompF} itself we also found SNPs in morbidostat samples. While \textit{ompC} was affected by a SNP causing a silent mutation in OmpC, the mutation in \textit{ompF} caused a missense mutation in OmpF. The missense mutation in OmpF could cause a change of the structure of the porin, which potentially affects the permeability of the outer membrane for cefepime.\\
We found another SNP in the promoter of \textit{fepA}. FepA is a gated porin transporting unspecific hydrophilic substances \cite{liu_permeability_1993}. The mutation in the promoter potentially repressed the expression of \textit{fepA} which may further decrease the membrane permeability for cefepime. 

\subsection{SNPs in the transcription machinery operon}
SNPs were identified targeting the transcription machinery in patient isolates and morbidostat samples. In one patient isolate we found that a mutation in \textit{rpoB} coding for the DNA-directed RNA polymerase subunit \textbeta. In a morbidostat sample we found a SNP targeting \textit{rpoD} coding for the RNA polymerase sigma factor. This strongly suggests that some process in the RNA transcription is affecting the cefepime susceptibility. In a preprint Samantha Palace \textit{et al.} reported that they found a SNP in \textit{rpoB} in three ceftriaxone (a third-generation cephalosporin) resistant \textit{Neisseria gonorrhoeae} isolates \cite{palace_rna_2019}. They were able to recreate resistance by integrating the mutation in a ceftriaxone susceptible \textit{Neisseria gonorrhoeae} strain, proving the importance of the mutation for ceftriaxone resistance \cite{palace_rna_2019}. The mutation in \textit{rpoB} that they identified in not the same as the one that we found. They also reported a mutation in \textit{rpoD} in two \textit{Neisseria gonorrhoeae} isolates and were able to recreate resistance for this mutation \cite{palace_rna_2019}. Interestingly, it is not understood yet, how mutations in RpoB or RpoD are impacting cephalosporin resistance. 
\subsection{SNPs affecting other operons}
We also identified SNPs in genes for which the evidence was less clear that they potentially impact cefepime susceptibility. For example, one SNP was found in \textit{ortT}. OrtT is a toxin causing membrane damage resulting in reduced growth and increased persistence during stress related to amino acid and DNA synthesis \cite{islam_orphan_2015}.  Cells in the persistent state are generally thought to be more tolerant to antibiotics \cite{islam_orphan_2015}. While it is difficult to hypothesize why a mutated OrtT was beneficial for the strain, it makes sense that the persistent state is beneficial if antibiotic pressure is applied.  \\
One SNP affected the ESBL gene coding for \textbeta-lactamase CTX-M-1 in a morbidostat sample. This caused a missense mutation in the ESBL. The resulting variant is not know in the literature, therefore, it is not possible to estimate whether it is potentially degrading \textbeta-lactams more effectively compared to the wild-type. 
\section{Conclusion}
We studied cefepime resistance evolution in ESBL \textit{E. coli} patient isolates sampled at the University Hospital of Basel and assembled a morbidostat, which we used to experimentally evolve cefepime resistance in ESBL \textit{E. coli} strains. The assembled morbidostat showed great functionality, increasing the MIC over 100-fold compared to the MIC at the beginning. The bioinformatic analysis of the patient isolates and samples from morbidostat experiments revealed several SNPs in the genomes of ESBL \textit{E. coli} which evolved resistance. Interestingly, some SNPs were found in the same gene in patient isolates and morbidostat samples, strongly suggesting their relevance for cefepime resistance. Two systems were targeted independently by SNPs. One system were the porins, in particular the porins OmpD, OmpF and their regulatory proteins OmpR and EnvZ. The SNPs found in the porin operons could imply, that the expression level of the porins is reduced, potentially increasing the impermeability of the outer membrane for cefepime. For the other system which was targeted by SNPs, the DNA-directed RNA polymerase subunit \textbeta \space and its RNA polymerase sigma factor, the potential connection to cefepime resistance is unknown. Since the system was targeted by SNPs in patient isolates and morbidostat samples, it is still likely that the identified genotypes are affecting cefepime susceptibility. \\
With our genomic study of cefepime resistance evolution we could identified new genotypes which may impact cefepime susceptibility. Our findings support the relevance of other resistance mechanisms than cefepime degradation by \textbeta-lactamases which was the main focus of cefepime resistance investigations so far.  
\section{Outlook}
Our goal for future morbidostat experiments is to eliminate contaminations and repeat the resistance evolution experiments with the ESBL \textit{E. coli} strains that we used for this thesis. In order to eliminate the contaminations we will take several additional precautions. We will change the design of the vials and replace the polytetrafluoroethylene inserts with the plastic tubes by septa and syringes. This will guarantee sealed vials and will also improve sampling, because the septa allow sampling with a syringe without opening the vial, reducing the risk of contaminations. We also realized that the injecting pumps were autoclavable, which was the only piece of hardware in contact with fluids, which we had not autoclaved before the morbidostat experiments. In the future we will autoclave the pumps which eliminates further potential sources of contaminations. When we will repeat the morbidostat experiments without contaminations we expect to identify SNPs in the same genes confirming our current findings. \\
We will also try to extend our patient isolate collection sampled at the University of Basel and we will analyze the additional isolates with the existing bioinformatic pipeline.  \\
To finally confirm the importance of the identified SNPs, we could try to insert some identified mutations in cefepime susceptible \textit{E. coli} strains and check if the created genotypes show changed cefepime susceptibility. If so, this would suggest that the identified SNP is associated with cefepime resistance evolution. Once the morbidostat experiments have been repeated, we could also use the morbidostat for studying resistance evolution of other bacteria and antibiotics.